<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>CIViC API Reference</title>

    <style>
      .highlight table td { padding: 5px; }
.highlight table pre { margin: 0; }
.highlight, .highlight .w {
  color: #f8f8f2;
  background-color: #272822;
}
.highlight .err {
  color: #151515;
  background-color: #ac4142;
}
.highlight .c, .highlight .cd, .highlight .cm, .highlight .c1, .highlight .cs {
  color: #505050;
}
.highlight .cp {
  color: #f4bf75;
}
.highlight .nt {
  color: #f4bf75;
}
.highlight .o, .highlight .ow {
  color: #d0d0d0;
}
.highlight .p, .highlight .pi {
  color: #d0d0d0;
}
.highlight .gi {
  color: #90a959;
}
.highlight .gd {
  color: #ac4142;
}
.highlight .gh {
  color: #6a9fb5;
  background-color: #151515;
  font-weight: bold;
}
.highlight .k, .highlight .kn, .highlight .kp, .highlight .kr, .highlight .kv {
  color: #aa759f;
}
.highlight .kc {
  color: #d28445;
}
.highlight .kt {
  color: #d28445;
}
.highlight .kd {
  color: #d28445;
}
.highlight .s, .highlight .sb, .highlight .sc, .highlight .sd, .highlight .s2, .highlight .sh, .highlight .sx, .highlight .s1 {
  color: #90a959;
}
.highlight .sr {
  color: #75b5aa;
}
.highlight .si {
  color: #8f5536;
}
.highlight .se {
  color: #8f5536;
}
.highlight .nn {
  color: #f4bf75;
}
.highlight .nc {
  color: #f4bf75;
}
.highlight .no {
  color: #f4bf75;
}
.highlight .na {
  color: #6a9fb5;
}
.highlight .m, .highlight .mf, .highlight .mh, .highlight .mi, .highlight .il, .highlight .mo, .highlight .mb, .highlight .mx {
  color: #90a959;
}
.highlight .ss {
  color: #90a959;
}
    </style>
    <link href="stylesheets/screen.css" rel="stylesheet" media="screen" />
    <link href="stylesheets/print.css" rel="stylesheet" media="print" />
      <script src="javascripts/all.js"></script>
  </head>

  <body class="index" data-languages="[]">
    <a href="#" id="nav-button">
      <span>
        NAV
        <img src="images/navbar.png" />
      </span>
    </a>
    <div class="tocify-wrapper">
      <img src="images/logo.png" />
        <div class="lang-selector">
        </div>
        <div class="search">
          <input type="text" class="search" id="input-search" placeholder="Search">
        </div>
        <ul class="search-results"></ul>
      <div id="toc">
      </div>
        <ul class="toc-footer">
            <li><a href='https://cvic.genome.wustl.edu'>Back to CIViC</a></li>
            <li><a href='https://github.com/tripit/slate'>Documentation Powered by Slate</a></li>
        </ul>
    </div>
    <div class="page-wrapper">
      <div class="dark-box"></div>
      <div class="content">
        <h1 id="introduction">Introduction</h1>

<p>CIViC provides a simple API that can be utilized over HTTP in the programming language of your choice or even at the command line.</p>

<p>The CIViC API attempts to be RESTful whenever appropriate which hopefully makes it intuitive to use. There are four main entities in the system that a user may interact with: Genes, Variants, Variant Groups, and Evidence Items. These entities form a hierarchy that is reflected in the API endpoints. A Gene has one or more Variants, each of which may have one or more Evidence Items. Variants may also be collected into Variant Groups.</p>

<p><strong>Disclaimer</strong>: While CIViC is in beta, the API is subject to change in ways that might break existing scripts. When we officially launch, we will version the API such that breaking or non-backwards compatible changes will not not affect current users.</p>

<h1 id="genes">Genes</h1>

<p>For most users, the primary entry point for the CIViC API will be the Genes endpoint. This endpoint allows a user to retrieve information about what genes are in CIViC as well as get lists of variants for specific genes.</p>

<h2 id="get-a-list-of-genes">Get a list of genes</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/genes
</code></pre>

<blockquote>
<p>The above command returns JSON structured like this:</p>
</blockquote>
<pre class="highlight json"><code><span class="p">[</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">1</span><span class="p">,</span><span class="w">
    </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"ALK"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="mi">238</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="s2">"ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines."</span><span class="p">,</span><span class="w">
    </span><span class="nt">"variants"</span><span class="p">:[</span><span class="w">
      </span><span class="p">{</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"F1174L"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">8</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_items"</span><span class="p">:{</span><span class="w">
          </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="mi">6</span><span class="p">,</span><span class="w">
          </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="mi">0</span><span class="p">,</span><span class="w">
          </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="mi">0</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="p">{</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"EML4-ALK L1152R"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">307</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_items"</span><span class="p">:{</span><span class="w">
          </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="mi">1</span><span class="p">,</span><span class="w">
          </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="mi">0</span><span class="p">,</span><span class="w">
          </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="mi">0</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">},</span><span class="w">
    </span><span class="p">],</span><span class="w">
    </span><span class="nt">"aliases"</span><span class="p">:[</span><span class="w">
      </span><span class="s2">"NBLST3"</span><span class="p">,</span><span class="w">
      </span><span class="s2">"CD246"</span><span class="w">
    </span><span class="p">],</span><span class="w">
</span><span class="p">]</span><span class="w">
</span></code></pre>

<p>This endpoint returns a listing of genes in CIViC that contain at least one evidence item. Returned gene listings contain variant ids which can be used to query for more detailed variant information.
This endpoint is <em>paginated</em> by default. You can use the <code class="prettyprint">count</code> and <code class="prettyprint">page</code> parameters to iterate through all the variants.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/genes</code></p>

<h3 id="query-parameters">Query Parameters</h3>

<table><thead>
<tr>
<th>Parameter</th>
<th>Default</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>page</td>
<td>1</td>
<td>Which page of results to return</td>
</tr>
<tr>
<td>count</td>
<td>25</td>
<td>How many genes to return on a single page</td>
</tr>
</tbody></table>

<h2 id="get-details-for-a-specific-gene">Get details for a specific gene.</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/genes/ALK?identifier_type<span class="o">=</span>entrez_symbol
</code></pre>

<blockquote>
<p>The above command returns JSON structured like this (truncated for clarity):</p>
</blockquote>
<pre class="highlight json"><code><span class="p">{</span><span class="w">
  </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">1</span><span class="p">,</span><span class="w">
  </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"ALK"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="mi">238</span><span class="p">,</span><span class="w">
  </span><span class="nt">"description"</span><span class="p">:</span><span class="s2">"ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines."</span><span class="p">,</span><span class="w">
  </span><span class="nt">"variants"</span><span class="p">:[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"EML4-ALK L1152R"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">307</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_items"</span><span class="p">:{</span><span class="w">
        </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="mi">1</span><span class="p">,</span><span class="w">
        </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="mi">0</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"EML4-ALK"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">5</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_items"</span><span class="p">:{</span><span class="w">
        </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="mi">3</span><span class="p">,</span><span class="w">
        </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="mi">1</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"EML4-ALK AMPLIFICATION"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">170</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_items"</span><span class="p">:{</span><span class="w">
        </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="mi">3</span><span class="p">,</span><span class="w">
        </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="mi">0</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"variant_groups"</span><span class="p">:[],</span><span class="w">
  </span><span class="nt">"aliases"</span><span class="p">:[</span><span class="w">
    </span><span class="s2">"NBLST3"</span><span class="p">,</span><span class="w">
    </span><span class="s2">"CD246"</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"sources"</span><span class="p">:[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"citation"</span><span class="p">:</span><span class="s2">"Rossi et al., 2014, Int. J. Oncol."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="s2">"24889366"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"source_url"</span><span class="p">:</span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/24889366"</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"citation"</span><span class="p">:</span><span class="s2">"Shaw et al., 2013, J. Clin. Oncol."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="s2">"23401436"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"source_url"</span><span class="p">:</span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/23401436"</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">]</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<p>This endpoint retrieves details about a specific gene.</p>

<p>Not that the default behavior of this endpoint is to use internal CIViC ids. If you want to use Entrez ids or gene symbols, you need to specify the identifier_type parameter.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edum/api/genes/:id</code></p>

<h3 id="url-parameters">URL Parameters</h3>

<table><thead>
<tr>
<th>Parameter</th>
<th>Valid Values</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>identifier_type</td>
<td><code class="prettyprint">entrez_id</code>, <code class="prettyprint">entrez_symbol</code>, <code class="prettyprint">civic_id</code></td>
<td>Type of gene identifier used in your query.</td>
</tr>
</tbody></table>

<h1 id="variants">Variants</h1>

<p>The variants endpoint allows users to enumerate all of the variants present in CIViC as well as retrieve more detailed information on a specific variant. A common use case would be to get a list of variants for a gene
using the genes endpoint and then fetching detailed information for each variant via the variants endpoint.</p>

<h2 id="get-a-list-of-variants">Get a list of variants</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/variants
</code></pre>

<blockquote>
<p>The above command returns JSON structured like this:</p>
</blockquote>
<pre class="highlight json"><code><span class="p">[</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">1</span><span class="p">,</span><span class="w">
    </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="s2">"ABL1"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="mi">4</span><span class="p">,</span><span class="w">
    </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="mi">25</span><span class="p">,</span><span class="w">
    </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"BCR-ABL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="s2">"The BCR-ABL fusion protein, commonly referred to as the Philadelphia chromosome, is one of the most studied fusion genes in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML). However, despite its ability to initiate disease in mice, its status as an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and ponatinib) have seen some success in delivering a tumor response."</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"variant_types"</span><span class="p">:[</span><span class="w">
    </span><span class="p">],</span><span class="w">
    </span><span class="nt">"coordinates"</span><span class="p">:{</span><span class="w">
      </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="s2">"22"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"start"</span><span class="p">:</span><span class="s2">"23522397"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"stop"</span><span class="p">:</span><span class="s2">"23632600"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="s2">"ENST00000305877.8"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="s2">"9"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"start2"</span><span class="p">:</span><span class="s2">"133729451"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"stop2"</span><span class="p">:</span><span class="s2">"133763063"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="s2">"ENST00000318560.5"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="mi">75</span><span class="p">,</span><span class="w">
      </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="s2">"GRCh37"</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="nt">"evidence_items"</span><span class="p">:[</span><span class="w">
      </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">213</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"EID213"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"description"</span><span class="p">:</span><span class="s2">"Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"disease"</span><span class="p">:{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">4</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"Chronic Myeloid Leukemia"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"doid"</span><span class="p">:</span><span class="s2">"8552"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"url"</span><span class="p">:</span><span class="s2">"http://www.disease-ontology.org/?id=DOID:8552"</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="nt">"citation"</span><span class="p">:</span><span class="s2">"Schmidt et al., 2014, Leukemia"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"source_url"</span><span class="p">:</span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/25212276"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="s2">"25212276"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"drugs"</span><span class="p">:[</span><span class="w">
        </span><span class="p">],</span><span class="w">
        </span><span class="nt">"rating"</span><span class="p">:</span><span class="mi">2</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_level"</span><span class="p">:</span><span class="s2">"C"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_type"</span><span class="p">:</span><span class="s2">"Diagnostic"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"clinical_significance"</span><span class="p">:</span><span class="s2">"Positive"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_direction"</span><span class="p">:</span><span class="s2">"Does Not Support"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_hgvs"</span><span class="p">:</span><span class="s2">"N/A"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_origin"</span><span class="p">:</span><span class="s2">"Somatic Mutation"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"drug_interaction_type"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"status"</span><span class="p">:</span><span class="s2">"accepted"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"type"</span><span class="p">:</span><span class="s2">"evidence"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">220</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"EID220"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"description"</span><span class="p">:</span><span class="s2">"The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"disease"</span><span class="p">:{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">4</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"Chronic Myeloid Leukemia"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"doid"</span><span class="p">:</span><span class="s2">"8552"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"url"</span><span class="p">:</span><span class="s2">"http://www.disease-ontology.org/?id=DOID:8552"</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="nt">"citation"</span><span class="p">:</span><span class="s2">"An et al., 2010, Leuk. Res."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"source_url"</span><span class="p">:</span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/20537386"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="s2">"20537386"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"drugs"</span><span class="p">:[</span><span class="w">
        </span><span class="p">],</span><span class="w">
        </span><span class="nt">"rating"</span><span class="p">:</span><span class="mi">4</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_level"</span><span class="p">:</span><span class="s2">"A"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_type"</span><span class="p">:</span><span class="s2">"Diagnostic"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"clinical_significance"</span><span class="p">:</span><span class="s2">"Positive"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_direction"</span><span class="p">:</span><span class="s2">"Supports"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_hgvs"</span><span class="p">:</span><span class="s2">"N/A"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_origin"</span><span class="p">:</span><span class="s2">"Somatic Mutation"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"drug_interaction_type"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"status"</span><span class="p">:</span><span class="s2">"accepted"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"type"</span><span class="p">:</span><span class="s2">"evidence"</span><span class="w">
      </span><span class="p">},</span><span class="w">
    </span><span class="p">],</span><span class="w">
  </span><span class="err">}</span><span class="w">
</span><span class="p">]</span><span class="w">
</span></code></pre>

<p>This endpoint returns a listing of variants in CIViC that contain at least one evidence item. This endpoint is <em>paginated</em> by default. You can use the <code class="prettyprint">count</code> and <code class="prettyprint">page</code> parameters to iterate through all the variants.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/variants</code></p>

<h3 id="query-parameters">Query Parameters</h3>

<table><thead>
<tr>
<th>Parameter</th>
<th>Default</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>page</td>
<td>1</td>
<td>Which page of results to return</td>
</tr>
<tr>
<td>count</td>
<td>25</td>
<td>How many variants to return on a single page</td>
</tr>
</tbody></table>

<h2 id="get-details-for-a-specific-variant">Get details for a specific variant.</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/variants/8
</code></pre>

<blockquote>
<p>The above command returns JSON structured like this (truncated for clarity):</p>
</blockquote>
<pre class="highlight json"><code><span class="w">  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">1</span><span class="p">,</span><span class="w">
    </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="s2">"ABL1"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="mi">4</span><span class="p">,</span><span class="w">
    </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="mi">25</span><span class="p">,</span><span class="w">
    </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"BCR-ABL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="s2">"The BCR-ABL fusion protein, commonly referred to as the Philadelphia chromosome, is one of the most studied fusion genes in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML). However, despite its ability to initiate disease in mice, its status as an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and ponatinib) have seen some success in delivering a tumor response."</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"variant_types"</span><span class="p">:[</span><span class="w">
    </span><span class="p">],</span><span class="w">
    </span><span class="nt">"coordinates"</span><span class="p">:{</span><span class="w">
      </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="s2">"22"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"start"</span><span class="p">:</span><span class="s2">"23522397"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"stop"</span><span class="p">:</span><span class="s2">"23632600"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="s2">"ENST00000305877.8"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="s2">"9"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"start2"</span><span class="p">:</span><span class="s2">"133729451"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"stop2"</span><span class="p">:</span><span class="s2">"133763063"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="s2">"ENST00000318560.5"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="mi">75</span><span class="p">,</span><span class="w">
      </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="s2">"GRCh37"</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="nt">"evidence_items"</span><span class="p">:[</span><span class="w">
      </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">213</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"EID213"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"description"</span><span class="p">:</span><span class="s2">"Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"disease"</span><span class="p">:{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">4</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"Chronic Myeloid Leukemia"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"doid"</span><span class="p">:</span><span class="s2">"8552"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"url"</span><span class="p">:</span><span class="s2">"http://www.disease-ontology.org/?id=DOID:8552"</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="nt">"citation"</span><span class="p">:</span><span class="s2">"Schmidt et al., 2014, Leukemia"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"source_url"</span><span class="p">:</span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/25212276"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="s2">"25212276"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"drugs"</span><span class="p">:[</span><span class="w">
        </span><span class="p">],</span><span class="w">
        </span><span class="nt">"rating"</span><span class="p">:</span><span class="mi">2</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_level"</span><span class="p">:</span><span class="s2">"C"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_type"</span><span class="p">:</span><span class="s2">"Diagnostic"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"clinical_significance"</span><span class="p">:</span><span class="s2">"Positive"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_direction"</span><span class="p">:</span><span class="s2">"Does Not Support"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_hgvs"</span><span class="p">:</span><span class="s2">"N/A"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_origin"</span><span class="p">:</span><span class="s2">"Somatic Mutation"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"drug_interaction_type"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"status"</span><span class="p">:</span><span class="s2">"accepted"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"type"</span><span class="p">:</span><span class="s2">"evidence"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">220</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"EID220"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"description"</span><span class="p">:</span><span class="s2">"The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"disease"</span><span class="p">:{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">4</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"Chronic Myeloid Leukemia"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"doid"</span><span class="p">:</span><span class="s2">"8552"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"url"</span><span class="p">:</span><span class="s2">"http://www.disease-ontology.org/?id=DOID:8552"</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="nt">"citation"</span><span class="p">:</span><span class="s2">"An et al., 2010, Leuk. Res."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"source_url"</span><span class="p">:</span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/20537386"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="s2">"20537386"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"drugs"</span><span class="p">:[</span><span class="w">
        </span><span class="p">],</span><span class="w">
        </span><span class="nt">"rating"</span><span class="p">:</span><span class="mi">4</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_level"</span><span class="p">:</span><span class="s2">"A"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_type"</span><span class="p">:</span><span class="s2">"Diagnostic"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"clinical_significance"</span><span class="p">:</span><span class="s2">"Positive"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_direction"</span><span class="p">:</span><span class="s2">"Supports"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_hgvs"</span><span class="p">:</span><span class="s2">"N/A"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_origin"</span><span class="p">:</span><span class="s2">"Somatic Mutation"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"drug_interaction_type"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"status"</span><span class="p">:</span><span class="s2">"accepted"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"type"</span><span class="p">:</span><span class="s2">"evidence"</span><span class="w">
      </span><span class="p">},</span><span class="w">
    </span><span class="p">],</span><span class="w">
  </span><span class="err">}</span><span class="w">
</span></code></pre>

<p>This endpoint retrieves details about a specific variant, given its internal CIViC id.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edum/api/variants/:id</code></p>

<h1 id="evidence-items">Evidence Items</h1>

<p>The evidence items endpoint allows users to enumerate all of the evidence items present in CIViC as well as retrieve more detailed information on a specific evidence item. </p>

<h2 id="get-a-list-of-evidence-items">Get a list of evidence items</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/evidence_items
</code></pre>

<blockquote>
<p>The above command returns JSON structured like this:</p>
</blockquote>
<pre class="highlight json"><code><span class="p">[</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">1</span><span class="p">,</span><span class="w">
    </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"EID1"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="s2">"JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL)."</span><span class="p">,</span><span class="w">
    </span><span class="nt">"disease"</span><span class="p">:{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">1</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"Lymphoid Leukemia"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"doid"</span><span class="p">:</span><span class="s2">"10747"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"url"</span><span class="p">:</span><span class="s2">"http://www.disease-ontology.org/?id=DOID:10747"</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="nt">"citation"</span><span class="p">:</span><span class="s2">"Levine et al., 2005, Blood"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"source_url"</span><span class="p">:</span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/16081687"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="s2">"16081687"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"drugs"</span><span class="p">:[</span><span class="w">
    </span><span class="p">],</span><span class="w">
    </span><span class="nt">"rating"</span><span class="p">:</span><span class="mi">4</span><span class="p">,</span><span class="w">
    </span><span class="nt">"evidence_level"</span><span class="p">:</span><span class="s2">"B"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"evidence_type"</span><span class="p">:</span><span class="s2">"Diagnostic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"clinical_significance"</span><span class="p">:</span><span class="s2">"Negative"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"evidence_direction"</span><span class="p">:</span><span class="s2">"Supports"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"variant_hgvs"</span><span class="p">:</span><span class="s2">"9:5073770-5073770 (G-\u003eT)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"variant_origin"</span><span class="p">:</span><span class="s2">"Somatic Mutation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"drug_interaction_type"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"status"</span><span class="p">:</span><span class="s2">"accepted"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="s2">"evidence"</span><span class="w">
  </span><span class="p">}</span><span class="w">
</span><span class="p">]</span><span class="w">
</span></code></pre>

<p>This endpoint returns a listing of evidence items in CIViC. This endpoint is <em>paginated</em> by default. You can use the <code class="prettyprint">count</code> and <code class="prettyprint">page</code> parameters to iterate through all the evidence items.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/evidence_items</code></p>

<h3 id="query-parameters">Query Parameters</h3>

<table><thead>
<tr>
<th>Parameter</th>
<th>Default</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>page</td>
<td>1</td>
<td>Which page of results to return</td>
</tr>
<tr>
<td>count</td>
<td>25</td>
<td>How many evidence items to return on a single page</td>
</tr>
</tbody></table>

<h2 id="get-details-for-a-specific-evidence-item">Get details for a specific evidence item.</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/evidence_items/1
</code></pre>

<blockquote>
<p>The above command returns JSON structured like this (truncated for clarity):</p>
</blockquote>
<pre class="highlight json"><code><span class="w">  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">1</span><span class="p">,</span><span class="w">
    </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"EID1"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="s2">"JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL)."</span><span class="p">,</span><span class="w">
    </span><span class="nt">"disease"</span><span class="p">:{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">1</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"Lymphoid Leukemia"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"doid"</span><span class="p">:</span><span class="s2">"10747"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"url"</span><span class="p">:</span><span class="s2">"http://www.disease-ontology.org/?id=DOID:10747"</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="nt">"citation"</span><span class="p">:</span><span class="s2">"Levine et al., 2005, Blood"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"source_url"</span><span class="p">:</span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/16081687"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="s2">"16081687"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"drugs"</span><span class="p">:[</span><span class="w">
    </span><span class="p">],</span><span class="w">
    </span><span class="nt">"rating"</span><span class="p">:</span><span class="mi">4</span><span class="p">,</span><span class="w">
    </span><span class="nt">"evidence_level"</span><span class="p">:</span><span class="s2">"B"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"evidence_type"</span><span class="p">:</span><span class="s2">"Diagnostic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"clinical_significance"</span><span class="p">:</span><span class="s2">"Negative"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"evidence_direction"</span><span class="p">:</span><span class="s2">"Supports"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"variant_hgvs"</span><span class="p">:</span><span class="s2">"9:5073770-5073770 (G-\u003eT)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"variant_origin"</span><span class="p">:</span><span class="s2">"Somatic Mutation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"drug_interaction_type"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"status"</span><span class="p">:</span><span class="s2">"accepted"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="s2">"evidence"</span><span class="w">
  </span><span class="p">}</span><span class="w">
</span></code></pre>

<p>This endpoint retrieves details about a specific evidence item, given its internal CIViC id.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edum/api/evidence_items/:id</code></p>

<h1 id="variant-groups">Variant Groups</h1>

<p>The variant groups endpoint allows users to enumerate all of the variant groups present in CIViC.</p>

<h2 id="get-a-list-of-variant-groups">Get a list of variant groups</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/variant_groups
</code></pre>

<blockquote>
<p>The above command returns JSON structured like this:</p>
</blockquote>
<pre class="highlight json"><code><span class="p">[</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">2</span><span class="p">,</span><span class="w">
    </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"KIT Exon 17"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"variants"</span><span class="p">:[</span><span class="w">
      </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">65</span><span class="p">,</span><span class="w">
        </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="s2">"KIT"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="mi">29</span><span class="p">,</span><span class="w">
        </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="mi">3815</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"D816V"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"description"</span><span class="p">:</span><span class="s2">"KIT D816V is a mutation observed in acute myeloid leukemia (AML). This variant has been linked to poorer prognosis and worse outcome in AML patients."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"type"</span><span class="p">:</span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_types"</span><span class="p">:[</span><span class="w">
        </span><span class="p">],</span><span class="w">
        </span><span class="nt">"coordinates"</span><span class="p">:{</span><span class="w">
          </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="s2">"4"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"start"</span><span class="p">:</span><span class="s2">"55599321"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"stop"</span><span class="p">:</span><span class="s2">"55599321"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="s2">"A"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="s2">"T"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="s2">"ENST00000288135.5"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
          </span><span class="nt">"start2"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
          </span><span class="nt">"stop2"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
          </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="kc">null</span><span class="p">,</span><span class="w">
          </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="mi">75</span><span class="p">,</span><span class="w">
          </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="s2">"GRCh37"</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">],</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="s2">"variant_group"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"lifecycle_actions"</span><span class="p">:{</span><span class="w">
      </span><span class="nt">"created"</span><span class="p">:{</span><span class="w">
        </span><span class="nt">"order"</span><span class="p">:</span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"timestamp"</span><span class="p">:</span><span class="s2">"2015-06-21T16:49:39.979Z"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"user"</span><span class="p">:{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">41</span><span class="p">,</span><span class="w">
          </span><span class="nt">"email"</span><span class="p">:</span><span class="s2">"nspies13@gmail.com"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"Nick Spies"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"last_seen_at"</span><span class="p">:</span><span class="s2">"2016-03-30T21:15:43.620Z"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"username"</span><span class="p">:</span><span class="s2">"NickSpies"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"role"</span><span class="p">:</span><span class="s2">"admin"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"avatar_url"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon\u0026r=pg\u0026s=32"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"avatars"</span><span class="p">:{</span><span class="w">
            </span><span class="nt">"x128"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon\u0026r=pg\u0026s=128"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"x64"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon\u0026r=pg\u0026s=64"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"x32"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon\u0026r=pg\u0026s=32"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"x14"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon\u0026r=pg\u0026s=14"</span><span class="w">
          </span><span class="p">},</span><span class="w">
          </span><span class="nt">"area_of_expertise"</span><span class="p">:</span><span class="s2">"Clinical Scientist"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"orcid"</span><span class="p">:</span><span class="s2">""</span><span class="p">,</span><span class="w">
          </span><span class="nt">"display_name"</span><span class="p">:</span><span class="s2">"NickSpies"</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">}</span><span class="w">
</span><span class="p">]</span><span class="w">
</span></code></pre>

<p>This endpoint returns a listing of variant groups in CIViC. This endpoint is <em>paginated</em> by default. You can use the <code class="prettyprint">count</code> and <code class="prettyprint">page</code> parameters to iterate through all the variant groups.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/variant_groups</code></p>

<h3 id="query-parameters">Query Parameters</h3>

<table><thead>
<tr>
<th>Parameter</th>
<th>Default</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>page</td>
<td>1</td>
<td>Which page of results to return</td>
</tr>
<tr>
<td>count</td>
<td>25</td>
<td>How many evidence items to return on a single page</td>
</tr>
</tbody></table>

<h1 id="community">Community</h1>

<p>One of the core features of CIViC is the community interaction it facilitates. As such, all community features such as comments and suggested changes are also available via the API.</p>

<p>The below examples will work with any of the core CIViC entities (Genes, Variants, Evidence Items, and Variant Groups).</p>

<h2 id="get-comments-on-a-particular-variant">Get comments on a particular variant</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/variants/3/comments
</code></pre>

<blockquote>
<p>The above command returns JSON structured like this:</p>
</blockquote>
<pre class="highlight json"><code><span class="p">[</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">459</span><span class="p">,</span><span class="w">
    </span><span class="nt">"text"</span><span class="p">:</span><span class="s2">"What is the actual genome location of E255K?  \n\nOne has to be very careful with this locus because there are at least two alternative transcripts that have an E at position 255 within their respective sequences, but because of alternative first coding exons, these are different positions in the protein sequence and genome.  Yamamoto, et al. (http://www.ncbi.nlm.nih.gov/pubmed/15194504) refer to E255K and cite this GenBank protein sequence as a reference: http://www.ncbi.nlm.nih.gov/protein/AAB60394.  Based on the position of E255K I believe it corresponds to the following transcript/amino acid positions.  \n\n* ABL1 ENST00000318560.5.v75 (E255K)\n* ABL1 ENST00000372348.2.v75 (E274K)\n* ABL1 NM_005157 (E255K)\n* ABL1 NM_007313 (E274K)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"title"</span><span class="p">:</span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"created_at"</span><span class="p">:</span><span class="s2">"2015-09-17T23:08:12.643Z"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"updated_at"</span><span class="p">:</span><span class="s2">"2015-09-17T23:08:12.643Z"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"user"</span><span class="p">:{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">15</span><span class="p">,</span><span class="w">
      </span><span class="nt">"email"</span><span class="p">:</span><span class="s2">"malachig@gmail.com"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"Malachi Griffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"last_seen_at"</span><span class="p">:</span><span class="s2">"2016-04-08T15:17:02.916Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"username"</span><span class="p">:</span><span class="s2">"MalachiGriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"role"</span><span class="p">:</span><span class="s2">"admin"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatar_url"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon\u0026r=pg\u0026s=32"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatars"</span><span class="p">:{</span><span class="w">
        </span><span class="nt">"x128"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon\u0026r=pg\u0026s=128"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x64"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon\u0026r=pg\u0026s=64"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x32"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon\u0026r=pg\u0026s=32"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x14"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon\u0026r=pg\u0026s=14"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"area_of_expertise"</span><span class="p">:</span><span class="s2">"Research Scientist"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"orcid"</span><span class="p">:</span><span class="s2">"0000-0002-6388-446X"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"display_name"</span><span class="p">:</span><span class="s2">"MalachiGriffith"</span><span class="w">
    </span><span class="p">},</span><span class="w">
  </span><span class="err">}</span><span class="p">,</span><span class="w">
</span><span class="p">]</span><span class="w">
</span></code></pre>

<p>This endpoint will return the discussion on a particular variant. Similar requests will work for other entities like genes and evidence items.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/variants/3/comments</code></p>

<h2 id="get-a-list-of-suggested-changes-for-an-evidence-item">Get a list of suggested changes for an evidence item</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/evidence_items/1253/suggested_changes
</code></pre>

<blockquote>
<p>The above command returns JSON structured like this:</p>
</blockquote>
<pre class="highlight json"><code><span class="p">[</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">1474</span><span class="p">,</span><span class="w">
    </span><span class="nt">"user"</span><span class="p">:{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">100</span><span class="p">,</span><span class="w">
      </span><span class="nt">"email"</span><span class="p">:</span><span class="s2">"damian.rieke@charite.de"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"Damian Rieke"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"last_seen_at"</span><span class="p">:</span><span class="s2">"2016-04-14T07:30:45.862Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"username"</span><span class="p">:</span><span class="s2">"DTRieke"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"role"</span><span class="p">:</span><span class="s2">"curator"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatar_url"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/baaffa3938cc82f434ca5561e34d3de9.png?d=identicon\u0026r=pg\u0026s=32"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatars"</span><span class="p">:{</span><span class="w">
        </span><span class="nt">"x128"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/baaffa3938cc82f434ca5561e34d3de9.png?d=identicon\u0026r=pg\u0026s=128"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x64"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/baaffa3938cc82f434ca5561e34d3de9.png?d=identicon\u0026r=pg\u0026s=64"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x32"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/baaffa3938cc82f434ca5561e34d3de9.png?d=identicon\u0026r=pg\u0026s=32"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x14"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/baaffa3938cc82f434ca5561e34d3de9.png?d=identicon\u0026r=pg\u0026s=14"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"area_of_expertise"</span><span class="p">:</span><span class="s2">"Clinical Scientist"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"orcid"</span><span class="p">:</span><span class="s2">"0000-0003-0027-7977"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"display_name"</span><span class="p">:</span><span class="s2">"DTRieke"</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="nt">"status"</span><span class="p">:</span><span class="s2">"applied"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"created_at"</span><span class="p">:</span><span class="s2">"2016-04-12T12:37:35.792Z"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"suggested_changes"</span><span class="p">:{</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:[</span><span class="w">
        </span><span class="s2">"Preclinical study of PF-06463922 in NSCLC in-vitro, in-vivo and in enzyme assays. PF-06463922 showed inhibition of kinase activity, cell viability and tumor growth in wild-type, G2032R and L2026M mutations in ROS1.The L2026M mutation was structurally (gatekeeper-mutation) predicted to emerge under crizotinib treatment and lead to crizotinib resistance in-vitro and in kinase assays."</span><span class="p">,</span><span class="w">
        </span><span class="s2">"Preclinical study of PF-06463922 in NSCLC in-vitro, in-vivo and in enzyme assays. PF-06463922 showed inhibition of kinase activity, cell viability and tumor growth in wild-type, G2032R and L2026M mutations in ROS1.The L2026M mutation was structurally (gatekeeper-mutation) predicted to emerge under crizotinib treatment and led to crizotinib resistance in-vitro and in kinase assays."</span><span class="w">
      </span><span class="p">]</span><span class="w">
    </span><span class="p">},</span><span class="w">
  </span><span class="err">}</span><span class="w">
</span><span class="p">]</span><span class="w">
</span></code></pre>

<p>This endpoint will return a list of suggested changes to a particular evidence item. Similar requests will work for other entities like variants and genes.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/evidence_items/1253/suggested_changes</code></p>

<h2 id="get-comments-on-a-single-suggested-change">Get comments on a single suggested change</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/evidence_items/1253/suggested_changes/1474/comments
</code></pre>

<blockquote>
<p>The above command returns JSON structured like this:</p>
</blockquote>
<pre class="highlight json"><code><span class="p">[</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">459</span><span class="p">,</span><span class="w">
    </span><span class="nt">"text"</span><span class="p">:</span><span class="s2">"What is the actual genome location of E255K?  \n\nOne has to be very careful with this locus because there are at least two alternative transcripts that have an E at position 255 within their respective sequences, but because of alternative first coding exons, these are different positions in the protein sequence and genome.  Yamamoto, et al. (http://www.ncbi.nlm.nih.gov/pubmed/15194504) refer to E255K and cite this GenBank protein sequence as a reference: http://www.ncbi.nlm.nih.gov/protein/AAB60394.  Based on the position of E255K I believe it corresponds to the following transcript/amino acid positions.  \n\n* ABL1 ENST00000318560.5.v75 (E255K)\n* ABL1 ENST00000372348.2.v75 (E274K)\n* ABL1 NM_005157 (E255K)\n* ABL1 NM_007313 (E274K)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"title"</span><span class="p">:</span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"created_at"</span><span class="p">:</span><span class="s2">"2015-09-17T23:08:12.643Z"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"updated_at"</span><span class="p">:</span><span class="s2">"2015-09-17T23:08:12.643Z"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"user"</span><span class="p">:{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="mi">15</span><span class="p">,</span><span class="w">
      </span><span class="nt">"email"</span><span class="p">:</span><span class="s2">"malachig@gmail.com"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="s2">"Malachi Griffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"last_seen_at"</span><span class="p">:</span><span class="s2">"2016-04-08T15:17:02.916Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"username"</span><span class="p">:</span><span class="s2">"MalachiGriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"role"</span><span class="p">:</span><span class="s2">"admin"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatar_url"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon\u0026r=pg\u0026s=32"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatars"</span><span class="p">:{</span><span class="w">
        </span><span class="nt">"x128"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon\u0026r=pg\u0026s=128"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x64"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon\u0026r=pg\u0026s=64"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x32"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon\u0026r=pg\u0026s=32"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x14"</span><span class="p">:</span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon\u0026r=pg\u0026s=14"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"area_of_expertise"</span><span class="p">:</span><span class="s2">"Research Scientist"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"orcid"</span><span class="p">:</span><span class="s2">"0000-0002-6388-446X"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"display_name"</span><span class="p">:</span><span class="s2">"MalachiGriffith"</span><span class="w">
    </span><span class="p">},</span><span class="w">
  </span><span class="err">}</span><span class="p">,</span><span class="w">
</span><span class="p">]</span><span class="w">
</span></code></pre>

<p>This endpoint will return the comment thread attached to a specific suggested change.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/evidence_items/1253/suggested_changes/1474/comments</code></p>

      </div>
      <div class="dark-box">
          <div class="lang-selector">
          </div>
      </div>
    </div>
  </body>
</html>
