<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>CIViC API Reference</title>

    <style>
      .highlight table td { padding: 5px; }
.highlight table pre { margin: 0; }
.highlight, .highlight .w {
  color: #f8f8f2;
  background-color: #272822;
}
.highlight .err {
  color: #272822;
  background-color: #f92672;
}
.highlight .c, .highlight .cd, .highlight .cm, .highlight .c1, .highlight .cs {
  color: #75715e;
}
.highlight .cp {
  color: #f4bf75;
}
.highlight .nt {
  color: #f4bf75;
}
.highlight .o, .highlight .ow {
  color: #f8f8f2;
}
.highlight .p, .highlight .pi {
  color: #f8f8f2;
}
.highlight .gi {
  color: #a6e22e;
}
.highlight .gd {
  color: #f92672;
}
.highlight .gh {
  color: #66d9ef;
  background-color: #272822;
  font-weight: bold;
}
.highlight .k, .highlight .kn, .highlight .kp, .highlight .kr, .highlight .kv {
  color: #ae81ff;
}
.highlight .kc {
  color: #fd971f;
}
.highlight .kt {
  color: #fd971f;
}
.highlight .kd {
  color: #fd971f;
}
.highlight .s, .highlight .sb, .highlight .sc, .highlight .sd, .highlight .s2, .highlight .sh, .highlight .sx, .highlight .s1 {
  color: #a6e22e;
}
.highlight .sr {
  color: #a1efe4;
}
.highlight .si {
  color: #cc6633;
}
.highlight .se {
  color: #cc6633;
}
.highlight .nn {
  color: #f4bf75;
}
.highlight .nc {
  color: #f4bf75;
}
.highlight .no {
  color: #f4bf75;
}
.highlight .na {
  color: #66d9ef;
}
.highlight .m, .highlight .mf, .highlight .mh, .highlight .mi, .highlight .il, .highlight .mo, .highlight .mb, .highlight .mx {
  color: #a6e22e;
}
.highlight .ss {
  color: #a6e22e;
}
    </style>
    <link href="stylesheets/screen.css" rel="stylesheet" media="screen" />
    <link href="stylesheets/print.css" rel="stylesheet" media="print" />
      <script src="javascripts/all.js"></script>
  </head>

  <body class="index" data-languages="[&quot;shell&quot;,&quot;json&quot;]">
    <a href="#" id="nav-button">
      <span>
        NAV
        <img src="images/navbar.png" alt="Navbar" />
      </span>
    </a>
    <div class="tocify-wrapper">
      <img src="images/logo.png" alt="Logo" />
        <div class="lang-selector">
              <a href="#" data-language-name="shell">Request</a>
              <a href="#" data-language-name="json">Response</a>
        </div>
        <div class="search">
          <input type="text" class="search" id="input-search" placeholder="Search">
        </div>
        <ul class="search-results"></ul>
      <div id="toc">
      </div>
        <ul class="toc-footer">
            <li><a href='https://civic.genome.wustl.edu'>Back to CIViC</a></li>
            <li><a href='https://github.com/tripit/slate'>Documentation Powered by Slate</a></li>
        </ul>
    </div>
    <div class="page-wrapper">
      <div class="dark-box"></div>
      <div class="content">
        <h1 id="introduction">Introduction</h1>

<p>CIViC provides a simple API that can be utilized over HTTP in the programming language of your choice or even at the command line.</p>

<p>The CIViC API attempts to be RESTful whenever appropriate which hopefully makes it intuitive to use. There are four main entities in the system that a user may interact with: Genes, Variants, Variant Groups, and Evidence Items. These entities form a hierarchy that is reflected in the API endpoints. A Gene has one or more Variants, each of which may have one or more Evidence Items. Variants may also be collected into Variant Groups.</p>

<p>This documentation provides example requests and responses for most endpoints. The responses are edited for brevity, but should include instances of each top level key for reference. You can toggle between viewing a request and its corresponding response using the tabs on the right of the page.</p>

          <h1 id="endpoint-types">Endpoint Types</h1>

<p>The API for consuming CIViC data can mostly be broken down into two different types of endpoints:
<strong>index</strong> and <strong>detail</strong>.</p>

<h2 id="index">Index</h2>

<blockquote>
<p>Example of a meta section for the genes index endpoint</p>
</blockquote>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/genes
</code></pre><pre class="highlight json"><code><span class="p">{</span><span class="w">
  </span><span class="nt">"_meta"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"current_page"</span><span class="p">:</span><span class="w"> </span><span class="mi">2</span><span class="p">,</span><span class="w">
      </span><span class="nt">"per_page"</span><span class="p">:</span><span class="w"> </span><span class="s2">"25"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"total_pages"</span><span class="p">:</span><span class="w"> </span><span class="mi">11</span><span class="p">,</span><span class="w">
      </span><span class="nt">"total_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">266</span><span class="p">,</span><span class="w">
      </span><span class="nt">"links"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"next"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://civic.genome.wustl.edu/api/genes?count=25&amp;page=3"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"previous"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://civic.genome.wustl.edu/api/genes?count=25&amp;page=1"</span><span class="w">
      </span><span class="p">}</span><span class="w">
   </span><span class="p">}</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<p>The index endpoints provide high level overview information about a collection of entities all at once and allow a user to page through all the entities in the system.
Index endpoints provide two top-level keys in their JSON structure: <code class="prettyprint">_meta</code> and <code class="prettyprint">records</code>.</p>

<p><strong>Meta</strong></p>

<p>As seen to the right, the <code class="prettyprint">_meta</code> section contains information about the total number of available records, the page size, and provides links to the <code class="prettyprint">next</code> and <code class="prettyprint">previous</code> page of results.
These links can be used to easily traverse all of the records in CIViC. A <code class="prettyprint">null</code> entry for <code class="prettyprint">next</code> (or <code class="prettyprint">previous</code>) indicates that you have reached the end of the collection.</p>

<p><strong>Records</strong></p>

<p>The <code class="prettyprint">records</code> section will contain the actual objects requested in the API call (ie: <code class="prettyprint">genes</code>, <code class="prettyprint">variants</code> or <code class="prettyprint">evidence_items</code>)</p>

<p>If you do not wish to use the links in the <code class="prettyprint">_meta</code> section, index endpoints also accept manual pagination parameters in the query string:</p>

<table><thead>
<tr>
<th>Parameter</th>
<th>Default</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>page</td>
<td>1</td>
<td>Which page of results to return</td>
</tr>
<tr>
<td>count</td>
<td>25</td>
<td>How many records to return on a single page</td>
</tr>
</tbody></table>

<h2 id="detail">Detail</h2>

<blockquote>
<p>Example of a detail response for the genes endpoint</p>
</blockquote>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/genes/19
</code></pre><pre class="highlight json"><code><span class="p">{</span><span class="w">
  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">19</span><span class="p">,</span><span class="w">
  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EGFR"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1956</span><span class="p">,</span><span class="w">
  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EGFR is widely recognized for its importance in cancer. Amplification and mutations have been shown to be driving events in many cancer types. Its role in non-small cell lung cancer, glioblastoma and basal-like breast cancers has spurred many research and drug development efforts. Tyrosine kinase inhibitors have shown efficacy in EGFR amplfied tumors, most notably gefitinib and erlotinib. Mutations in EGFR have been shown to confer resistance to these drugs, particularly the variant T790M, which has been functionally characterized as a resistance marker for both of these drugs. The later generation TKI's have seen some success in treating these resistant cases, and targeted sequencing of the EGFR locus has become a common practice in treatment of non-small cell lung cancer. \nOverproduction of ligands is another possible mechanism of activation of EGFR. ERBB ligands include EGF, TGF-a, AREG, EPG, BTC, HB-EGF, EPR and NRG1-4 (for detailed information please refer to the respective ligand section). In ligand-activated cancers, Cetuximab appears to be more effective than tyrosine-kinase inhibitors (Arteaga et. al.)."</span><span class="p">,</span><span class="w">
  </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"S492R"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">453</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">4</span><span class="p">,</span><span class="w">
        </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"3' UTR MUTATION"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">253</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
        </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"K467T"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">455</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">2</span><span class="p">,</span><span class="w">
        </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"aliases"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="s2">"HER1"</span><span class="p">,</span><span class="w">
    </span><span class="s2">"ERBB1"</span><span class="p">,</span><span class="w">
    </span><span class="s2">"ERBB"</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"gene"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"variant_groups"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">12</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EGFR TKI Resistance"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EGFR pathway activation is a nearly ubiquitous hallmark of cancer. Many tyrosine kinase inhibitors have been developed to target EGFR pathway activity. One such inhibitor, erlotinib, has demonstrated efficacy in an EGFR over-active setting. However, the T790M missense mutation has shown to confer resistance to this inhibitor in cell lines and case studies.  "</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">34</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EGFR"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">19</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1956</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"T790M"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EGFR T790M was one of the very first mutations recognized to confer resistance to targeted therapies in non-small cell lung cancer. While successful in amplified EGFR, the efficacy of the first and second generation TKI's (erlotinib, gefitinib, neratinib) in treating patients harboring this mutation before treatment is notably lower. This lack of efficacy can likely be to blame for the poorer prognosis for patients with this mutation as compared to patients with wildtype EGFR or other types of EGFR mutations. Approximately half of EGFR mutant tumors with acquired resistance to TKI inhibition have been shown to harbor this mutation, implicating it as a mechanism of acquired therapy resistence. A third generation TKI (osimertinib) has been approved for the treatment of EGFR T790M mutant NSCLC. Patients positive for T790M in a plasma-based test have similar outcomes like those with tumor biopsy testing."</span><span class="p">,</span><span class="w">
          </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
            </span><span class="p">}</span><span class="w">
          </span><span class="p">],</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">19</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="w">
          </span><span class="p">},</span><span class="w">
          </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"7"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55249071"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55249071"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"C"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"T"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000275493.2"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant_group"</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"lifecycle_actions"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"last_modified"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"order"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
      </span><span class="nt">"timestamp"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2016-01-07T09:04:18.614Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"user"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">100</span><span class="p">,</span><span class="w">
        </span><span class="nt">"email"</span><span class="p">:</span><span class="w"> </span><span class="s2">"damian.rieke@charite.de"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Damian Rieke"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"last_seen_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2016-12-12T13:23:11.578Z"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"username"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DTRieke"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"role"</span><span class="p">:</span><span class="w"> </span><span class="s2">"editor"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"avatar_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/baaffa3938cc82f434ca5561e34d3de9.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"avatars"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
          </span><span class="nt">"x128"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/baaffa3938cc82f434ca5561e34d3de9.png?d=identicon&amp;r=pg&amp;s=128"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x64"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/baaffa3938cc82f434ca5561e34d3de9.png?d=identicon&amp;r=pg&amp;s=64"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x32"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/baaffa3938cc82f434ca5561e34d3de9.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x14"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/baaffa3938cc82f434ca5561e34d3de9.png?d=identicon&amp;r=pg&amp;s=14"</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="nt">"area_of_expertise"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Clinical Scientist"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"orcid"</span><span class="p">:</span><span class="w"> </span><span class="s2">"0000-0003-0027-7977"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DTRieke"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"created_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-12-14T08:37:49.264Z"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
        </span><span class="nt">"twitter_handle"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"facebook_profile"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"linkedin_profile"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"bio"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"featured_expert"</span><span class="p">:</span><span class="w"> </span><span class="kc">false</span><span class="p">,</span><span class="w">
        </span><span class="nt">"accepted_license"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"signup_complete"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">},</span><span class="w">
  </span><span class="err">}</span><span class="p">,</span><span class="w">
  </span><span class="nt">"sources"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">26</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Epidermal growth factor receptor targeting in cancer: a review of trends and strategies."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"citation"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Yewale et al., 2013, Biomaterials"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"23953842"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"source_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/23953842"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"open_access"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"pmc_id"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"publication_date"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"year"</span><span class="p">:</span><span class="w"> </span><span class="mi">2013</span><span class="p">,</span><span class="w">
        </span><span class="nt">"month"</span><span class="p">:</span><span class="w"> </span><span class="mi">11</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"journal"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Biomaterials"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"full_journal_title"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Biomaterials"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"status"</span><span class="p">:</span><span class="w"> </span><span class="s2">"fully curated"</span><span class="w">
    </span><span class="p">},</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"errors"</span><span class="p">:</span><span class="w"> </span><span class="p">{}</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<p>Detail endpoints return the full CIViC record for a single entity, specified explicitly by id.
In addition to the high level overview information returned by the index endpoint, this complete record will include information about data provenance, timestamps, and secondary relationships.
Unlike the index endpoints, detail endpoints do not have separate <code class="prettyprint">_meta</code> and <code class="prettyprint">records</code> sections as only a single record is returned.</p>

          <h1 id="genes">Genes</h1>

<p>For most users, the primary entry point for the CIViC API will be the Genes endpoint. This endpoint allows a user to retrieve information about what genes are in CIViC as well as get lists of variants for specific genes.</p>

<h2 id="get-a-list-of-genes">Get a list of genes</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/genes
</code></pre><pre class="highlight json"><code><span class="p">{</span><span class="w">
  </span><span class="nt">"_meta"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"current_page"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
    </span><span class="nt">"per_page"</span><span class="p">:</span><span class="w"> </span><span class="mi">25</span><span class="p">,</span><span class="w">
    </span><span class="nt">"total_pages"</span><span class="p">:</span><span class="w"> </span><span class="mi">11</span><span class="p">,</span><span class="w">
    </span><span class="nt">"total_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">267</span><span class="p">,</span><span class="w">
    </span><span class="nt">"links"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"next"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://civic.genome.wustl.edu/api/genes?count=25&amp;page=2"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"previous"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="nt">"records"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">238</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DEL4-11"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">550</span><span class="p">,</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CLTC-ALK"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">520</span><span class="p">,</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EML4-ALK  V1180L"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">528</span><span class="p">,</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">5</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">},</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"aliases"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="s2">"NBLST3"</span><span class="p">,</span><span class="w">
        </span><span class="s2">"CD246"</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"gene"</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">2</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"AKT1"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">207</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"AKT1, also referred to as protein kinase B, is a known oncogene. AKT activation relies on the PI3K pathway, and is recognized as a critical node in the pathway. The E17 hotspot is the most characterized of AKT1 mutations, and has been shown to result in activation of the protein. Mutations in AKT1 have also been shown to confer resistance to allosteric kinase inhibitors in vitro."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Q79K"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">169</span><span class="p">,</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"aliases"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="s2">"RAC-ALPHA"</span><span class="p">,</span><span class="w">
        </span><span class="s2">"PKB"</span><span class="p">,</span><span class="w">
        </span><span class="s2">"AKT"</span><span class="p">,</span><span class="w">
        </span><span class="s2">"PKB-ALPHA"</span><span class="p">,</span><span class="w">
        </span><span class="s2">"RAC"</span><span class="p">,</span><span class="w">
        </span><span class="s2">"CWS6"</span><span class="p">,</span><span class="w">
        </span><span class="s2">"PRKBA"</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"gene"</span><span class="w">
    </span><span class="p">},</span><span class="w">
  </span><span class="p">]</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<p>This endpoint returns a listing of genes in CIViC that contain at least one evidence item. Returned gene listings contain variant ids which can be used to query for more detailed variant information.
This is an index style endpoint and is <em>paginated</em> by default. You can use the <code class="prettyprint">count</code> and <code class="prettyprint">page</code> parameters to iterate through all the variants.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/genes</code></p>

<h3 id="query-parameters">Query Parameters</h3>

<table><thead>
<tr>
<th>Parameter</th>
<th>Default</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>page</td>
<td>1</td>
<td>Which page of results to return</td>
</tr>
<tr>
<td>count</td>
<td>25</td>
<td>How many genes to return on a single page</td>
</tr>
</tbody></table>

<h2 id="get-details-for-a-specific-gene">Get details for a specific gene</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/genes/ALK?identifier_type<span class="o">=</span>entrez_symbol
</code></pre><pre class="highlight json"><code><span class="p">{</span><span class="w">
  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">238</span><span class="p">,</span><span class="w">
  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK amplifications, fusions and mutations have been shown to be driving events in non-small cell lung cancer. While crizontinib has demonstrated efficacy in treating the amplification, mutations in ALK have been shown to confer resistance to current tyrosine kinase inhibitors. Second-generation TKI's have seen varied success in treating these resistant cases, and the HSP90 inhibitor 17-AAG has been shown to be cytostatic in ALK-altered cell lines."</span><span class="p">,</span><span class="w">
  </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EML4-ALK L1152R"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">307</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
        </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"F1245C"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">549</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">2</span><span class="p">,</span><span class="w">
        </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">},</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"aliases"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="s2">"NBLST3"</span><span class="p">,</span><span class="w">
    </span><span class="s2">"CD246"</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"gene"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"variant_groups"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">11</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK Fusions"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK fusion positive non-small cell lung cancer (NSCLC) is treated as its own subset of NSCLC. Many ALK fusions that have been seen as recurrent in cancer serve to increase the activity of the ALK oncogene relative to normal cells. While EML4 is the most common fusion partner, other 5' partners have been observed. The EML4-ALK fusion has shown sensitivity to targeted tyrosine kinase inhibitors such as crizotinib. "</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">520</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">238</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CLTC-ALK"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The t(2;17)(p23;q23) translocation results in the CLTC-ALK fusion protein, the most common ALK fusion observed in diffuse large B cell lymphoma (DLBCL). ALK-rearranged DLBCL is less responsive to CHOP chemotherapy. Preclinical work indicates that CLTC-ALK DLBCL is responsive to ALK inhibition in cell lines and mouse models, and two case studies show short response followed by progression when heavily pretreated and advanced ALK-positive DLBCL is treated with crizotinib (one case CLTC-ALK, one case an unknown ALK fusion)."</span><span class="p">,</span><span class="w">
          </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">120</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"transcript_fusion"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Transcript Fusion"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001886"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A feature fusion where the deletion brings together transcript regions."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"</span><span class="w">
            </span><span class="p">}</span><span class="w">
          </span><span class="p">],</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
          </span><span class="p">},</span><span class="w">
          </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"17"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"57697219"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"57768072"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000269122.3"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"29415640"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"29446394"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000389048.3"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">514</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">238</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"RANBP2-ALK"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
          </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">120</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"transcript_fusion"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Transcript Fusion"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001886"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A feature fusion where the deletion brings together transcript regions."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"</span><span class="w">
            </span><span class="p">}</span><span class="w">
          </span><span class="p">],</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
          </span><span class="p">},</span><span class="w">
          </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"109335937"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"109375004"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000283195.6"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"29415640"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"29446394"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000389048.3"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">},</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant_group"</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"lifecycle_actions"</span><span class="p">:</span><span class="w"> </span><span class="p">{},</span><span class="w">
  </span><span class="nt">"sources"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">2</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK in lung cancer: past, present, and future."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"citation"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Shaw et al., 2013, J. Clin. Oncol."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"23401436"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"source_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/23401436"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"open_access"</span><span class="p">:</span><span class="w"> </span><span class="kc">true</span><span class="p">,</span><span class="w">
      </span><span class="nt">"pmc_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PMC4209068"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"publication_date"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"year"</span><span class="p">:</span><span class="w"> </span><span class="mi">2013</span><span class="p">,</span><span class="w">
        </span><span class="nt">"month"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
        </span><span class="nt">"day"</span><span class="p">:</span><span class="w"> </span><span class="mi">10</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"journal"</span><span class="p">:</span><span class="w"> </span><span class="s2">"J. Clin. Oncol."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"full_journal_title"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Journal of clinical oncology : official journal of the American Society of Clinical Oncology"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"status"</span><span class="p">:</span><span class="w"> </span><span class="s2">"fully curated"</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"errors"</span><span class="p">:</span><span class="w"> </span><span class="p">{}</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<p>This endpoint retrieves details about a specific gene.</p>

<p>Note that the default behavior of this endpoint is to use internal CIViC ids. If you want to use Entrez ids or gene symbols, you need to specify the identifier_type parameter.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/genes/:id</code></p>

<h3 id="url-parameters">URL Parameters</h3>

<table><thead>
<tr>
<th>Parameter</th>
<th>Valid Values</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>identifier_type</td>
<td><code class="prettyprint">entrez_id</code>, <code class="prettyprint">entrez_symbol</code>, <code class="prettyprint">civic_id</code></td>
<td>Type of gene identifier used in your query.</td>
</tr>
</tbody></table>

<h2 id="get-several-genes-at-once-via-an-indentifier">Get several genes at once via an indentifier</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/genes/TP53,BRAF?identifier_type<span class="o">=</span>entrez_symbol
</code></pre><pre class="highlight json"><code><span class="p">[</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">5</span><span class="p">,</span><span class="w">
    </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"BRAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">673</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"BRAF mutations are found to be recurrent in many cancer types. Of these, the mutation of valine 600 to glutamic acid (V600E) is the most prevalent. V600E has been determined to be an activating mutation, and cells that harbor it, along with other V600 mutations are sensitive to the BRAF inhibitor dabrafenib. It is also common to use MEK inhibition as a substitute for BRAF inhibitors, and the MEK inhibitor trametinib has seen some success in BRAF mutant melanomas. BRAF mutations have also been correlated with poor prognosis in many cancer types, although there is at least one study that questions this conclusion in papillary thyroid cancer."</span><span class="p">,</span><span class="w">
    </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
      </span><span class="p">{</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"D594G"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">611</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
          </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
          </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
          </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="p">{</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"L597V"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">585</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
          </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
          </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
          </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">},</span><span class="w">
    </span><span class="p">],</span><span class="w">
    </span><span class="nt">"aliases"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
      </span><span class="s2">"RAFB1"</span><span class="p">,</span><span class="w">
      </span><span class="s2">"NS7"</span><span class="p">,</span><span class="w">
      </span><span class="s2">"BRAF1"</span><span class="p">,</span><span class="w">
      </span><span class="s2">"B-RAF1"</span><span class="w">
    </span><span class="p">],</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"gene"</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">45</span><span class="p">,</span><span class="w">
    </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TP53"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">7157</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TP53 mutations are universal across cancer types. Loss of tumor suppressors is most recognized by large deleterious events, such as frameshift mutations, or premature stop codons. In TP53 however, many of the observed mutations in cancer are found to be single nucleotide variants, or missense mutations. These variants are also very broadly distributed throughout the gene, not localizing in any particular hotspot. While a large proportion of cancer genomics research is focused on somatic variants, TP53 is also of note in the germline. Germline TP53 mutations are the hallmark of Li-Fraumeni syndrome, and many (both germline and somatic) have been found to have prognostic impact on patient outcomes. The significance of many polymorphisms in susceptibility and prognosis of disease is still very much up for debate."</span><span class="p">,</span><span class="w">
    </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
      </span><span class="p">{</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"P72R"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">531</span><span class="p">,</span><span class="w">
        </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
          </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
          </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
          </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">},</span><span class="w">
    </span><span class="p">],</span><span class="w">
    </span><span class="nt">"aliases"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
      </span><span class="s2">"TRP53"</span><span class="p">,</span><span class="w">
      </span><span class="s2">"P53"</span><span class="p">,</span><span class="w">
      </span><span class="s2">"LFS1"</span><span class="p">,</span><span class="w">
      </span><span class="s2">"BCC7"</span><span class="w">
    </span><span class="p">],</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"gene"</span><span class="w">
  </span><span class="p">}</span><span class="w">
</span><span class="p">]</span><span class="w">
</span></code></pre>
<p>The endpoint can be used to fetch overview information for multiple genes at the same time. It supports the same <code class="prettyprint">identifier_type</code>s as the detail endpoint.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/genes/TP53,BRAF?identifier_type=entrez_symbol</code></p>

<h3 id="query-parameters">Query Parameters</h3>

<table><thead>
<tr>
<th>Parameter</th>
<th>Valid Values</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>identifier_type</td>
<td><code class="prettyprint">entrez_id</code>, <code class="prettyprint">entrez_symbol</code>, <code class="prettyprint">civic_id</code></td>
<td>Type of gene identifier used in your query.</td>
</tr>
</tbody></table>

          <h1 id="variants">Variants</h1>

<p>The variants endpoint allows users to enumerate all of the variants present in CIViC as well as retrieve more detailed information on a specific variant. A common use case would be to get a list of variants for a gene
using the genes endpoint and then fetching detailed information for each variant via the variants endpoint.</p>

<h2 id="get-a-list-of-variants">Get a list of variants</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/variants
</code></pre><pre class="highlight json"><code><span class="p">{</span><span class="w">
  </span><span class="nt">"_meta"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"current_page"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
    </span><span class="nt">"per_page"</span><span class="p">:</span><span class="w"> </span><span class="mi">25</span><span class="p">,</span><span class="w">
    </span><span class="nt">"total_pages"</span><span class="p">:</span><span class="w"> </span><span class="mi">30</span><span class="p">,</span><span class="w">
    </span><span class="nt">"total_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">729</span><span class="p">,</span><span class="w">
    </span><span class="nt">"links"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"next"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://civic.genome.wustl.edu/api/variants?count=25&amp;page=2"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"previous"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="nt">"records"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
      </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ABL1"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">4</span><span class="p">,</span><span class="w">
      </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">25</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"BCR-ABL"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The BCR-ABL fusion protein, commonly referred to as the Philadelphia chromosome, is one of the most studied fusion genes in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML). However, despite its ability to initiate disease in mice, its status as an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and ponatinib) have seen some success in delivering a tumor response."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">7</span><span class="p">,</span><span class="w">
        </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"22"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"23522397"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"23632600"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000305877.8"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"9"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133729451"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133763063"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000318560.5"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
        </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">28</span><span class="p">,</span><span class="w">
      </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CEBPA"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">15</span><span class="p">,</span><span class="w">
      </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1050</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"N-TERMINAL FRAME SHIFT"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CEBPA N-terminal frame shift mutations that result in a premature stop codon are associated with cytogenetically normal acute myeloid leukemia (CN-AML). CEBPA mutations are associated with a favorable prognosis; however, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications. N-terminal frame shift mutations within amino acids 1-120 have been shown to disrupt expression of the full-length 42kDa isoform while maintaining expression of a 30kDa isoform (translated from the AUG at full-length protein isoform amino acid M120). This 30kDa isoform exhibits dominant negative activity."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">134</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"frameshift_truncation"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Frameshift Truncation"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001910"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A frameshift variant that causes the translational reading frame to be shortened relative to the reference feature."</span><span class="p">,</span><span class="w">
          </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001910"</span><span class="w">
        </span><span class="p">},</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">2</span><span class="p">,</span><span class="w">
        </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"19"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"33792961"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"33793470"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000498907.2"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
        </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">]</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<p>This endpoint returns a listing of variants in CIViC that contain at least one evidence item. This index style endpoint is <em>paginated</em> by default. You can use the <code class="prettyprint">count</code> and <code class="prettyprint">page</code> parameters or the <code class="prettyprint">previous</code> and <code class="prettyprint">next</code> links to iterate through all the variants.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/variants</code></p>

<h3 id="query-parameters">Query Parameters</h3>

<table><thead>
<tr>
<th>Parameter</th>
<th>Default</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>page</td>
<td>1</td>
<td>Which page of results to return</td>
</tr>
<tr>
<td>count</td>
<td>25</td>
<td>How many variants to return on a single page</td>
</tr>
</tbody></table>

<h2 id="get-details-for-a-specific-variant">Get details for a specific variant.</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/variants/8
</code></pre><pre class="highlight json"><code><span class="p">{</span><span class="w">
  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">8</span><span class="p">,</span><span class="w">
  </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
  </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">238</span><span class="p">,</span><span class="w">
  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"F1174L"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells."</span><span class="p">,</span><span class="w">
  </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1271</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EID1271"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"In a Phase I trial (NCT00939770) of the drug crizotinib in pediatric patients, a subset of 11 neuroblastoma patients were characterized for mutations in ALK. Of these 11 patients, 4 had the F1174L mutation. Among these 4 patients, stable disease (SD) was observed in one, and progressive disease (PD) was observed in 3 others after administration of crizotinib. In contrast, in the remaining 7 patients with mutations different than F1174L, 4 had PD, 2 had SD and one had complete response as best response after initiation of crizotinib."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"disease"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">13</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Neuroblastoma"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Neuroblastoma"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"doid"</span><span class="p">:</span><span class="w"> </span><span class="s2">"769"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.disease-ontology.org/?id=DOID:769"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"drugs"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">12</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Crizotinib"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"pubchem_id"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"rating"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_level"</span><span class="p">:</span><span class="w"> </span><span class="s2">"B"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Predictive"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"clinical_significance"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Sensitivity"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_direction"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Does Not Support"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variant_origin"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Somatic Mutation"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"drug_interaction_type"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"status"</span><span class="p">:</span><span class="w"> </span><span class="s2">"accepted"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"open_change_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"evidence"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">859</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Safety and activity of crizotinib for paediatric patients with refractory solid tumours or anaplastic large-cell lymphoma: a Children's Oncology Group phase 1 consortium study."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"citation"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Moss et al., 2013, Lancet Oncol."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"23598171"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"source_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/23598171"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"open_access"</span><span class="p">:</span><span class="w"> </span><span class="kc">true</span><span class="p">,</span><span class="w">
        </span><span class="nt">"pmc_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PMC3730818"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"publication_date"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
          </span><span class="nt">"year"</span><span class="p">:</span><span class="w"> </span><span class="mi">2013</span><span class="p">,</span><span class="w">
          </span><span class="nt">"month"</span><span class="p">:</span><span class="w"> </span><span class="mi">5</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="nt">"journal"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Lancet Oncol."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"full_journal_title"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The Lancet. Oncology"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"status"</span><span class="p">:</span><span class="w"> </span><span class="s2">"fully curated"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"variant_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">8</span><span class="w">
    </span><span class="p">},</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"29443695"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"29443695"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"G"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"T"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000389048.3"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
    </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="nt">"variant_groups"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Crizotinib Resistance"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The ALK oncogene has long been considered a driving factor in non-small cell lung cancer (NSCLC). The targeted tyrosine kinase inhibitor criztonib has shown to be effective in ALK-mutant NSCLC. However, in patients that have shown acquired resistance to crizotinib, missense mutations in the tyrosine kinase domain have shown to drive this resistance. "</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">6</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ALK"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">238</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EML4-ALK C1156Y"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, C1156Y has been shown to confer resistance to crizotinib. Case reports indicate that secondary mutations can modulate drug sensitivity. EML4-ALK C1156Y/L1196M maintained crizotinib resistance while the lorlatinib resistant combination EML4-ALK C1156Y/L1198F re-sensitized the tumor to crizotinib treatment."</span><span class="p">,</span><span class="w">
          </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
           </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">120</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"transcript_fusion"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Transcript Fusion"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001886"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A feature fusion where the deletion brings together transcript regions."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"</span><span class="w">
            </span><span class="p">}</span><span class="w">
          </span><span class="p">],</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
          </span><span class="p">},</span><span class="w">
          </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"29445258"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"29445258"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"C"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"T"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000389048.3"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">},</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant_group"</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"variant_aliases"</span><span class="p">:</span><span class="w"> </span><span class="p">[],</span><span class="w">
  </span><span class="nt">"hgvs_expressions"</span><span class="p">:</span><span class="w"> </span><span class="p">[],</span><span class="w">
  </span><span class="nt">"clinvar_entries"</span><span class="p">:</span><span class="w"> </span><span class="p">[],</span><span class="w">
  </span><span class="nt">"lifecycle_actions"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"last_reviewed"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"order"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
      </span><span class="nt">"timestamp"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2016-05-03T15:11:18.404Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"user"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">6</span><span class="p">,</span><span class="w">
        </span><span class="nt">"email"</span><span class="p">:</span><span class="w"> </span><span class="s2">"kkrysiak@genome.wustl.edu"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Kilannin Krysiak"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"last_seen_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2017-01-04T16:05:02.406Z"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"username"</span><span class="p">:</span><span class="w"> </span><span class="s2">"kkrysiak"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"role"</span><span class="p">:</span><span class="w"> </span><span class="s2">"admin"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"avatar_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/17180f9afc9f7f04fff97197c1ee5cb6.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"avatars"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
          </span><span class="nt">"x128"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/17180f9afc9f7f04fff97197c1ee5cb6.png?d=identicon&amp;r=pg&amp;s=128"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x64"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/17180f9afc9f7f04fff97197c1ee5cb6.png?d=identicon&amp;r=pg&amp;s=64"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x32"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/17180f9afc9f7f04fff97197c1ee5cb6.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x14"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/17180f9afc9f7f04fff97197c1ee5cb6.png?d=identicon&amp;r=pg&amp;s=14"</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="nt">"area_of_expertise"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Research Scientist"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"orcid"</span><span class="p">:</span><span class="w"> </span><span class="s2">"0000-0002-6299-9230"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"kkrysiak"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"created_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-02-26T04:14:20.953Z"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
        </span><span class="nt">"twitter_handle"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
        </span><span class="nt">"facebook_profile"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
        </span><span class="nt">"linkedin_profile"</span><span class="p">:</span><span class="w"> </span><span class="s2">"kilannin-krysiak-69047819"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"bio"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Dr. Krysiak is a staff scientist at the McDonnell Genome Institute at Washington University School of Medicine where she is involved in the comprehensive genomic analysis of cancer patient cohorts and n-of-1 studies. She received her PhD in Molecular Genetics and Genomics at Washington University in St. Louis where she focused on the genetics of myelodysplastic syndrome through advanced flow cytometry techniques, primary cell culture and mouse models. She is a founding member of the CIViC team, helping to define the CIViC data model, and a leading content curator and feature development consultant."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"featured_expert"</span><span class="p">:</span><span class="w"> </span><span class="kc">true</span><span class="p">,</span><span class="w">
        </span><span class="nt">"accepted_license"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"signup_complete"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="nt">"sources"</span><span class="p">:</span><span class="w"> </span><span class="p">[],</span><span class="w">
  </span><span class="nt">"errors"</span><span class="p">:</span><span class="w"> </span><span class="p">{}</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<p>This endpoint retrieves details about a specific variant, given its internal CIViC id.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/variants/:id</code></p>

          <h1 id="evidence-items">Evidence Items</h1>

<p>The evidence items endpoint allows users to enumerate all of the evidence items present in CIViC as well as retrieve more detailed information on a specific evidence item. This is the endpoint that should be used to obtain a complete listing of every evidence item in CIViC.</p>

<h2 id="get-a-list-of-evidence-items">Get a list of evidence items</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/evidence_items
</code></pre><pre class="highlight json"><code><span class="p">{</span><span class="w">
  </span><span class="nt">"_meta"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"current_page"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
    </span><span class="nt">"per_page"</span><span class="p">:</span><span class="w"> </span><span class="mi">25</span><span class="p">,</span><span class="w">
    </span><span class="nt">"total_pages"</span><span class="p">:</span><span class="w"> </span><span class="mi">69</span><span class="p">,</span><span class="w">
    </span><span class="nt">"total_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1722</span><span class="p">,</span><span class="w">
    </span><span class="nt">"links"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"next"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://civic.genome.wustl.edu/api/evidence_items?count=25&amp;page=2"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"previous"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="nt">"records"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EID1"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL)."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"disease"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Lymphoid Leukemia"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Lymphoid Leukemia"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"doid"</span><span class="p">:</span><span class="w"> </span><span class="s2">"10747"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.disease-ontology.org/?id=DOID:10747"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"drugs"</span><span class="p">:</span><span class="w"> </span><span class="p">[],</span><span class="w">
      </span><span class="nt">"rating"</span><span class="p">:</span><span class="w"> </span><span class="mi">4</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_level"</span><span class="p">:</span><span class="w"> </span><span class="s2">"B"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Diagnostic"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"clinical_significance"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Negative"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_direction"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Supports"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variant_origin"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Somatic Mutation"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"drug_interaction_type"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"status"</span><span class="p">:</span><span class="w"> </span><span class="s2">"accepted"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"open_change_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"evidence"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">51</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The JAK2V617F activating mutation occurs in chronic myelomonocytic leukemia and acute myeloid leukemia, but not in acute lymphoblastic leukemia or chronic lymphocytic leukemia."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"citation"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Levine et al., 2005, Blood"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"16081687"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"source_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/16081687"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"open_access"</span><span class="p">:</span><span class="w"> </span><span class="kc">true</span><span class="p">,</span><span class="w">
        </span><span class="nt">"pmc_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PMC1895066"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"publication_date"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
          </span><span class="nt">"year"</span><span class="p">:</span><span class="w"> </span><span class="mi">2005</span><span class="p">,</span><span class="w">
          </span><span class="nt">"month"</span><span class="p">:</span><span class="w"> </span><span class="mi">11</span><span class="p">,</span><span class="w">
          </span><span class="nt">"day"</span><span class="p">:</span><span class="w"> </span><span class="mi">15</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="nt">"journal"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Blood"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"full_journal_title"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Blood"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"status"</span><span class="p">:</span><span class="w"> </span><span class="s2">"fully curated"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"variant_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">64</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">2</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EID2"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GIST tumors harboring PDGFRA D842V mutation are more likely to be benign than malignant."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"disease"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">2</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Gastrointestinal Stromal Tumor"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Gastrointestinal Stromal Tumor"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"doid"</span><span class="p">:</span><span class="w"> </span><span class="s2">"9253"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.disease-ontology.org/?id=DOID:9253"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"drugs"</span><span class="p">:</span><span class="w"> </span><span class="p">[],</span><span class="w">
      </span><span class="nt">"rating"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_level"</span><span class="p">:</span><span class="w"> </span><span class="s2">"B"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Diagnostic"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"clinical_significance"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Negative"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"evidence_direction"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Supports"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variant_origin"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Somatic Mutation"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"drug_interaction_type"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"status"</span><span class="p">:</span><span class="w"> </span><span class="s2">"accepted"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"open_change_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"evidence"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">52</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A great majority of GISTs with PDGFRA mutations represent gastric tumors of low or no malignant potential."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"citation"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Lasota et al., 2004, Lab. Invest."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"15146165"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"source_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/15146165"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"open_access"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"pmc_id"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"publication_date"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
          </span><span class="nt">"year"</span><span class="p">:</span><span class="w"> </span><span class="mi">2004</span><span class="p">,</span><span class="w">
          </span><span class="nt">"month"</span><span class="p">:</span><span class="w"> </span><span class="mi">7</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="nt">"journal"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Lab. Invest."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"full_journal_title"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Laboratory investigation; a journal of technical methods and pathology"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"status"</span><span class="p">:</span><span class="w"> </span><span class="s2">"fully curated"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"variant_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">99</span><span class="w">
    </span><span class="p">},</span><span class="w">
  </span><span class="p">]</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<p>This endpoint returns a listing of evidence items in CIViC. This index style endpoint is <em>paginated</em> by default. You can use the <code class="prettyprint">count</code> and <code class="prettyprint">page</code> parameters or the <code class="prettyprint">previous</code> and <code class="prettyprint">next</code> links to iterate through all the evidence items.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/evidence_items</code></p>

<h3 id="query-parameters">Query Parameters</h3>

<table><thead>
<tr>
<th>Parameter</th>
<th>Default</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>page</td>
<td>1</td>
<td>Which page of results to return</td>
</tr>
<tr>
<td>count</td>
<td>25</td>
<td>How many variants to return on a single page</td>
</tr>
</tbody></table>

<h2 id="get-details-for-a-specific-evidence-item">Get details for a specific evidence item</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/evidence_items/512
</code></pre><pre class="highlight json"><code><span class="p">{</span><span class="w">
  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">512</span><span class="p">,</span><span class="w">
  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"EID512"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Nonsense mutations in SMARCA4 were associated with small cell carcinoma of the ovary, hypercalcemic type in 10/10 tumor samples, and were rarely found in other solid tumor types catalogued by TCGA."</span><span class="p">,</span><span class="w">
  </span><span class="nt">"disease"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">253</span><span class="p">,</span><span class="w">
    </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Small Cell Carcinoma Of The Ovary Hypercalcemic Type"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Small Cell Carcinoma Of The Ovary Hypercalcemic Type"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"doid"</span><span class="p">:</span><span class="w"> </span><span class="s2">"7651"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.disease-ontology.org/?id=DOID:7651"</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="nt">"drugs"</span><span class="p">:</span><span class="w"> </span><span class="p">[],</span><span class="w">
  </span><span class="nt">"rating"</span><span class="p">:</span><span class="w"> </span><span class="mi">5</span><span class="p">,</span><span class="w">
  </span><span class="nt">"evidence_level"</span><span class="p">:</span><span class="w"> </span><span class="s2">"B"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"evidence_type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Diagnostic"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"clinical_significance"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Positive"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"evidence_direction"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Supports"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"variant_origin"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Somatic Mutation"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"drug_interaction_type"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
  </span><span class="nt">"status"</span><span class="p">:</span><span class="w"> </span><span class="s2">"accepted"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"open_change_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
  </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"evidence"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">316</span><span class="p">,</span><span class="w">
    </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Recurrent SMARCA4 mutations in small cell carcinoma of the ovary."</span><span class="p">,</span><span class="w">
    </span><span class="nt">"citation"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Jelinic et al., 2014, Nat. Genet."</span><span class="p">,</span><span class="w">
    </span><span class="nt">"pubmed_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"24658004"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"source_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.ncbi.nlm.nih.gov/pubmed/24658004"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"open_access"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"pmc_id"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"publication_date"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"year"</span><span class="p">:</span><span class="w"> </span><span class="mi">2014</span><span class="p">,</span><span class="w">
      </span><span class="nt">"month"</span><span class="p">:</span><span class="w"> </span><span class="mi">5</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="nt">"journal"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Nat. Genet."</span><span class="p">,</span><span class="w">
    </span><span class="nt">"full_journal_title"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Nature genetics"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"status"</span><span class="p">:</span><span class="w"> </span><span class="s2">"fully curated"</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="nt">"variant_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">217</span><span class="p">,</span><span class="w">
  </span><span class="nt">"errors"</span><span class="p">:</span><span class="w"> </span><span class="p">{},</span><span class="w">
  </span><span class="nt">"lifecycle_actions"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"accepted"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"order"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
      </span><span class="nt">"timestamp"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-08-05T18:47:08.046Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"user"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">41</span><span class="p">,</span><span class="w">
        </span><span class="nt">"email"</span><span class="p">:</span><span class="w"> </span><span class="s2">"nspies13@gmail.com"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Nick Spies"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"last_seen_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2016-12-02T22:15:09.110Z"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"username"</span><span class="p">:</span><span class="w"> </span><span class="s2">"NickSpies"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"role"</span><span class="p">:</span><span class="w"> </span><span class="s2">"admin"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"avatar_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"avatars"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
          </span><span class="nt">"x128"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=128"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x64"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=64"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x32"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x14"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=14"</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="nt">"area_of_expertise"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Research Scientist"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"orcid"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
        </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"NickSpies"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"created_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-06-12T18:13:16.508Z"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
        </span><span class="nt">"twitter_handle"</span><span class="p">:</span><span class="w"> </span><span class="s2">"@NickSpies13"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"facebook_profile"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"linkedin_profile"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"bio"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Nick Spies is a staff analyst at the McDonnell Genome Institute and an MD student at Washington University School of Medicine. He has made substantial contributions to the development of genome analysis tools and resources at the Genome Institute including the Drug-Gene Interaction Database. He is a founding member of the CIViC team, helping to define the CIViC data model, and a leading content curator and a feature development consultant."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"featured_expert"</span><span class="p">:</span><span class="w"> </span><span class="kc">true</span><span class="p">,</span><span class="w">
        </span><span class="nt">"accepted_license"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"signup_complete"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="nt">"fields_with_pending_changes"</span><span class="p">:</span><span class="w"> </span><span class="p">{}</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<p>This endpoint retrieves details about a specific evidence item, given its internal CIViC id.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/evidence_items/:id</code></p>

          <h1 id="variant-groups">Variant Groups</h1>

<p>The variant groups endpoint allows users to enumerate all of the variant groups in CIViC and view their membership.</p>

<h2 id="get-a-list-of-variant-groups">Get a list of variant groups</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/variant_groups
</code></pre><pre class="highlight json"><code><span class="p">{</span><span class="w">
  </span><span class="nt">"_meta"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"current_page"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
    </span><span class="nt">"per_page"</span><span class="p">:</span><span class="w"> </span><span class="mi">20</span><span class="p">,</span><span class="w">
    </span><span class="nt">"total_pages"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
    </span><span class="nt">"total_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">20</span><span class="p">,</span><span class="w">
    </span><span class="nt">"links"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"next"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"previous"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="nt">"records"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Imatinib Resistance"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ABL1"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">4</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">25</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"BCR-ABL E255K"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."</span><span class="p">,</span><span class="w">
          </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">120</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"transcript_fusion"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Transcript Fusion"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001886"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A feature fusion where the deletion brings together transcript regions."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"</span><span class="w">
            </span><span class="p">},</span><span class="w">
          </span><span class="p">],</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
          </span><span class="p">},</span><span class="w">
          </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"9"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133738363"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133738363"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"G"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000318560.5"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">101</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">38</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">5156</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"I843DEL"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."</span><span class="p">,</span><span class="w">
          </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">107</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"inframe_deletion"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Inframe Deletion"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001822"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"An inframe non synonymous variant that deletes bases from the coding sequence."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"</span><span class="w">
            </span><span class="p">}</span><span class="w">
          </span><span class="p">],</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">2</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
          </span><span class="p">},</span><span class="w">
          </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"4"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152095"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152097"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ATC"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000257290.5"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">102</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">38</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">5156</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DI842-843VM"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. The DI842-843VM variant is the result of a double point mutation. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."</span><span class="p">,</span><span class="w">
          </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
            </span><span class="p">}</span><span class="w">
          </span><span class="p">],</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
          </span><span class="p">},</span><span class="w">
          </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"4"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152093"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152097"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ACATC"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCATG"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000257290.5"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">98</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">38</span><span class="p">,</span><span class="w">
          </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">5156</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"D842I"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."</span><span class="p">,</span><span class="w">
          </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
            </span><span class="p">}</span><span class="w">
          </span><span class="p">],</span><span class="w">
          </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
            </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
            </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
          </span><span class="p">},</span><span class="w">
          </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
            </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"4"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152092"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152093"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GA"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"AT"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000257290.5"</span><span class="p">,</span><span class="w">
            </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
            </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
            </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
          </span><span class="p">}</span><span class="w">
        </span><span class="p">},</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant_group"</span><span class="w">
    </span><span class="p">},</span><span class="w">
  </span><span class="p">]</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<p>This endpoint returns a listing of variant groups in CIViC. This index style endpoint is <em>paginated</em> by default. You can use the <code class="prettyprint">count</code> and <code class="prettyprint">page</code> parameters or the <code class="prettyprint">previous</code> and <code class="prettyprint">next</code> links to iterate through all the variant groups.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/variant_groups</code></p>

<h3 id="query-parameters">Query Parameters</h3>

<table><thead>
<tr>
<th>Parameter</th>
<th>Default</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>page</td>
<td>1</td>
<td>Which page of results to return</td>
</tr>
<tr>
<td>count</td>
<td>25</td>
<td>How many variants to return on a single page</td>
</tr>
</tbody></table>

<h2 id="get-details-for-a-specific-variant-group">Get details for a specific variant group</h2>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/variant_groups/1
</code></pre><pre class="highlight json"><code><span class="p">{</span><span class="w">
  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Imatinib Resistance"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "</span><span class="p">,</span><span class="w">
  </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
    </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
      </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ABL1"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">4</span><span class="p">,</span><span class="w">
      </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">25</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"BCR-ABL E255K"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">120</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"transcript_fusion"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Transcript Fusion"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001886"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A feature fusion where the deletion brings together transcript regions."</span><span class="p">,</span><span class="w">
          </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
          </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">],</span><span class="w">
      </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
        </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
        </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"9"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133738363"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133738363"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"G"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000318560.5"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
        </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"variant_groups"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
        </span><span class="p">{</span><span class="w">
          </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
          </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Imatinib Resistance"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "</span><span class="p">,</span><span class="w">
          </span><span class="nt">"variants"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">2</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ABL1"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">4</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">25</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"BCR-ABL T334I"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I (aka T334I) has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and ponatinib) specific to BCR-ABL have shown efficacy in treating resistant cases."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
                </span><span class="p">{</span><span class="w">
                  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
                </span><span class="p">},</span><span class="w">
                </span><span class="p">{</span><span class="w">
                  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">120</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"transcript_fusion"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Transcript Fusion"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001886"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A feature fusion where the deletion brings together transcript regions."</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"</span><span class="w">
                </span><span class="p">}</span><span class="w">
              </span><span class="p">],</span><span class="w">
              </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
                </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
                </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
              </span><span class="p">},</span><span class="w">
              </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"9"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133748283"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133748283"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"C"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"T"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000372348.2"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
              </span><span class="p">}</span><span class="w">
            </span><span class="p">},</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ABL1"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">4</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">25</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"BCR-ABL E255K"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
                </span><span class="p">{</span><span class="w">
                  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
                </span><span class="p">},</span><span class="w">
                </span><span class="p">{</span><span class="w">
                  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">120</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"transcript_fusion"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Transcript Fusion"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001886"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A feature fusion where the deletion brings together transcript regions."</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"</span><span class="w">
                </span><span class="p">}</span><span class="w">
              </span><span class="p">],</span><span class="w">
              </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
                </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
                </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
              </span><span class="p">},</span><span class="w">
              </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"9"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133738363"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133738363"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"G"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000318560.5"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
              </span><span class="p">}</span><span class="w">
            </span><span class="p">},</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">98</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">38</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">5156</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"D842I"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
                </span><span class="p">{</span><span class="w">
                  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
                </span><span class="p">}</span><span class="w">
              </span><span class="p">],</span><span class="w">
              </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
                </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
                </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
              </span><span class="p">},</span><span class="w">
              </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"4"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152092"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152093"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GA"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"AT"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000257290.5"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
              </span><span class="p">}</span><span class="w">
            </span><span class="p">},</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">99</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">38</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">5156</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"D842V"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. Exogenous expression of the A842V mutation resulted in constitutive tyrosine phosphorylation of PDGFRA in the absence of ligand in 293T cells and cytokine-independent proliferation of the IL-3-dependent Ba/F3 cell line, both evidence that this is an activating mutation. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
                </span><span class="p">{</span><span class="w">
                  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
                </span><span class="p">}</span><span class="w">
              </span><span class="p">],</span><span class="w">
              </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">6</span><span class="p">,</span><span class="w">
                </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
                </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
              </span><span class="p">},</span><span class="w">
              </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"4"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152093"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152093"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"T"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000257290.5"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
              </span><span class="p">}</span><span class="w">
            </span><span class="p">},</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">100</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">38</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">5156</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"D842Y"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
                </span><span class="p">{</span><span class="w">
                  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
                </span><span class="p">}</span><span class="w">
              </span><span class="p">],</span><span class="w">
              </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
                </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
                </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
              </span><span class="p">},</span><span class="w">
              </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"4"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152092"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152092"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"G"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"T"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000257290.5"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
              </span><span class="p">}</span><span class="w">
            </span><span class="p">},</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">101</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">38</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">5156</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"I843DEL"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
                </span><span class="p">{</span><span class="w">
                  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">107</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"inframe_deletion"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Inframe Deletion"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001822"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"An inframe non synonymous variant that deletes bases from the coding sequence."</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"</span><span class="w">
                </span><span class="p">}</span><span class="w">
              </span><span class="p">],</span><span class="w">
              </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">2</span><span class="p">,</span><span class="w">
                </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
                </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
              </span><span class="p">},</span><span class="w">
              </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"4"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152095"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152097"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ATC"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000257290.5"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
              </span><span class="p">}</span><span class="w">
            </span><span class="p">},</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">102</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">38</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">5156</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DI842-843VM"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. The DI842-843VM variant is the result of a double point mutation. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
                </span><span class="p">{</span><span class="w">
                  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
                </span><span class="p">}</span><span class="w">
              </span><span class="p">],</span><span class="w">
              </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
                </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
                </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
              </span><span class="p">},</span><span class="w">
              </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"4"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152093"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"55152097"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ACATC"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCATG"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000257290.5"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
              </span><span class="p">}</span><span class="w">
            </span><span class="p">},</span><span class="w">
            </span><span class="p">{</span><span class="w">
              </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">241</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ABL1"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"gene_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">4</span><span class="p">,</span><span class="w">
              </span><span class="nt">"entrez_id"</span><span class="p">:</span><span class="w"> </span><span class="mi">25</span><span class="p">,</span><span class="w">
              </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"BCR-ABL F317L"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"BCR-ABL F317L, like the similar BCR-ABL T315I mutation, is becoming a common clinical marker for resistance to front-line therapies in CML. It has been shown to confer resistance to dasatinib, but responds well to ponatinib and other second generation inhibitors."</span><span class="p">,</span><span class="w">
              </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant"</span><span class="p">,</span><span class="w">
              </span><span class="nt">"variant_types"</span><span class="p">:</span><span class="w"> </span><span class="p">[</span><span class="w">
                </span><span class="p">{</span><span class="w">
                  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">47</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"missense_variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Missense Variant"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001583"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"</span><span class="w">
                </span><span class="p">},</span><span class="w">
                </span><span class="p">{</span><span class="w">
                  </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">120</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"transcript_fusion"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Transcript Fusion"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"so_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"SO:0001886"</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"A feature fusion where the deletion brings together transcript regions."</span><span class="p">,</span><span class="w">
                  </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"</span><span class="w">
                </span><span class="p">}</span><span class="w">
              </span><span class="p">],</span><span class="w">
              </span><span class="nt">"evidence_items"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"accepted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">3</span><span class="p">,</span><span class="w">
                </span><span class="nt">"rejected_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
                </span><span class="nt">"submitted_count"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="w">
              </span><span class="p">},</span><span class="w">
              </span><span class="nt">"coordinates"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
                </span><span class="nt">"chromosome"</span><span class="p">:</span><span class="w"> </span><span class="s2">"9"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133748288"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop"</span><span class="p">:</span><span class="w"> </span><span class="s2">"133748288"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"T"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"variant_bases"</span><span class="p">:</span><span class="w"> </span><span class="s2">"C"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ENST00000318560.5"</span><span class="p">,</span><span class="w">
                </span><span class="nt">"chromosome2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"start2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"stop2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"representative_transcript2"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
                </span><span class="nt">"ensembl_version"</span><span class="p">:</span><span class="w"> </span><span class="mi">75</span><span class="p">,</span><span class="w">
                </span><span class="nt">"reference_build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GRCh37"</span><span class="w">
              </span><span class="p">}</span><span class="w">
            </span><span class="p">}</span><span class="w">
          </span><span class="p">],</span><span class="w">
          </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant_group"</span><span class="w">
        </span><span class="p">}</span><span class="w">
      </span><span class="p">]</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">],</span><span class="w">
  </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"variant_group"</span><span class="p">,</span><span class="w">
  </span><span class="nt">"lifecycle_actions"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
    </span><span class="nt">"created"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"order"</span><span class="p">:</span><span class="w"> </span><span class="mi">1</span><span class="p">,</span><span class="w">
      </span><span class="nt">"timestamp"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-06-21T16:49:38.943Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"user"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">41</span><span class="p">,</span><span class="w">
        </span><span class="nt">"email"</span><span class="p">:</span><span class="w"> </span><span class="s2">"nspies13@gmail.com"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Nick Spies"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"last_seen_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2016-12-02T22:15:09.110Z"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"username"</span><span class="p">:</span><span class="w"> </span><span class="s2">"NickSpies"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"role"</span><span class="p">:</span><span class="w"> </span><span class="s2">"admin"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"avatar_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"avatars"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
          </span><span class="nt">"x128"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=128"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x64"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=64"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x32"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
          </span><span class="nt">"x14"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=14"</span><span class="w">
        </span><span class="p">},</span><span class="w">
        </span><span class="nt">"area_of_expertise"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Research Scientist"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"orcid"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
        </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"NickSpies"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"created_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-06-12T18:13:16.508Z"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
        </span><span class="nt">"twitter_handle"</span><span class="p">:</span><span class="w"> </span><span class="s2">"@NickSpies13"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"facebook_profile"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"linkedin_profile"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"bio"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Nick Spies is a staff analyst at the McDonnell Genome Institute and an MD student at Washington University School of Medicine. He has made substantial contributions to the development of genome analysis tools and resources at the Genome Institute including the Drug-Gene Interaction Database. He is a founding member of the CIViC team, helping to define the CIViC data model, and a leading content curator and a feature development consultant."</span><span class="p">,</span><span class="w">
        </span><span class="nt">"featured_expert"</span><span class="p">:</span><span class="w"> </span><span class="kc">true</span><span class="p">,</span><span class="w">
        </span><span class="nt">"accepted_license"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
        </span><span class="nt">"signup_complete"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
      </span><span class="p">}</span><span class="w">
    </span><span class="p">}</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="nt">"sources"</span><span class="p">:</span><span class="w"> </span><span class="p">[]</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<p>This endpoint retrieves details about a specific variant group, given its internal CIViC id.</p>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET https://civic.genome.wustl.edu/api/variant_groups/:id</code></p>

          <h1 id="linking">Linking</h1>

<p>For convenience, the CIViC API provides a method for linking to entities on the front end website using only a single identifier.
This allows external integrations to build links to CIViC without needing an entire hierarchy of ids.
The API supports two link formats, documented below, that behave identically.</p>

<p>These routes will return a <code class="prettyprint">404</code> response if the provided ID is not found. Otherwise, they will return a <code class="prettyprint">302</code> redirect to the canonical frontend link for the entity in question.</p>

<blockquote>
<p>To link to the CIViC page for the Entrez Gene 1956 (EGFR) with the resource style format:
<code class="prettyprint">
https://civic.genome.wustl.edu/links/entrez_id/1956
</code></p>
</blockquote>

<p><strong>Resource Style Links:</strong></p>

<p>Resource style links take the form of <code class="prettyprint">/links/:entity_type/:entity_id</code>.</p>

<blockquote>
<p>To link to the CIViC page for the variant with ID 12 (BRAF V600E) with the query parameter format:
<code class="prettyprint">
https://civic.genome.wustl.edu/links?idtype=variant&amp;id=12
</code></p>
</blockquote>

<p><strong>Query Parameter Links:</strong></p>

<p>Query parameter style links take the form of <code class="prettyprint">/links?idtype=:entity_type&amp;id=:entity_id</code></p>

<p>CIViC supports the following entity and id types for generating shortned links:</p>

<table><thead>
<tr>
<th>Entity Type</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>gene</td>
<td>CIViC Gene ID</td>
</tr>
<tr>
<td>entrez_id</td>
<td>Entrez Gene ID</td>
</tr>
<tr>
<td>variant</td>
<td>CIViC Variant ID</td>
</tr>
<tr>
<td>evidence</td>
<td>CIViC Evidence ID</td>
</tr>
</tbody></table>

<p>For more natural seeming resource based endpoints, the plural versions of the primary CViC entity types are also supported (<code class="prettyprint">genes</code>, <code class="prettyprint">variants</code>, <code class="prettyprint">evidence_items</code>) and behave identically to their singular counterparts.</p>

          <h1 id="metadata-endpoints">Metadata endpoints</h1>

<p>All core CIViC entities support several metadata endpoints that allow users to view discussion, revision history, and moderation history.</p>

<p>These endpoints can be accessed by appending an additional path segment onto a detail view URL.
For example, to see the discussion thread for the Variant with CIViC id 5, you could access the endpoint:</p>

<p><code class="prettyprint">https://civic.genome.wustl.edu/api/variants/5/comments</code></p>

<p>Additionally, due to the hierarchical nature of CIViC entities, you can append the type of a child entity to the detail view of a parent entity in order to view the children.</p>

<p>For example, you could get all of the <code class="prettyprint">variants</code> for a specific <code class="prettyprint">gene</code> by querying <code class="prettyprint">https://civic.genome.wustl.edu/api/genes/:gene_id/variants</code>
or all the <code class="prettyprint">evidence_items</code> for a specific <code class="prettyprint">variant</code> at the path <code class="prettyprint">https://civic.genome.wustl.edu/api/variants/:variant_id/evidence_items</code>.</p>

<h3 id="endpoint-types">Endpoint Types</h3>
<pre class="highlight shell"><code>curl https://civic.genome.wustl.edu/api/genes/49/comments
</code></pre><pre class="highlight json"><code><span class="p">[</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">247</span><span class="p">,</span><span class="w">
    </span><span class="nt">"text"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Note that there appears to be some confusion in the literature about the coordinates and nomenclature for WT1 mutations and the selection of representative transcripts.  This may stem in part from the fact that The WT1 gene encodes numerous protein isoforms. Two translational start sites have been identified, a standard methionine and a non-AUG leucine, which adds 68 amino-terminal residues (http://genesdev.cshlp.org/content/12/20/3217.full).  This confusion is propagated by the use of ambiguous terms \"Exon 7\" and \"Exon 9\" mutations in the literature, often used without additional information to help reduce ambiguity.  To help clarify these variants, here are some representative primer sequences used to amplify \"Exon 7\" and \"Exon 9\" of WT1 (http://www.ncbi.nlm.nih.gov/pubmed/19221039).\n\n    &gt;7F\n    CTCCAGTGCTCACTCTCCCTC\n    &gt;7R\n    CCTTAGCAGTGTGAGAGCCTG\n    &gt;9F\n    GTGAGGCAGATGCAGACATTG\n    &gt;9R\n    AGCCACGCACTATTCCTTCTC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"title"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"created_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-09-03T00:52:11.617Z"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"updated_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-09-03T00:52:11.617Z"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"user"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">15</span><span class="p">,</span><span class="w">
      </span><span class="nt">"email"</span><span class="p">:</span><span class="w"> </span><span class="s2">"malachig@gmail.com"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Malachi Griffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"last_seen_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2017-01-05T21:29:19.166Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"username"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MalachiGriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"role"</span><span class="p">:</span><span class="w"> </span><span class="s2">"admin"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatar_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatars"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"x128"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=128"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x64"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=64"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x32"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x14"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=14"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"area_of_expertise"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Research Scientist"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"orcid"</span><span class="p">:</span><span class="w"> </span><span class="s2">"0000-0002-6388-446X"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MalachiGriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"created_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-02-26T22:25:34.692Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://genome.wustl.edu/people/individual/malachi-griffith/"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"twitter_handle"</span><span class="p">:</span><span class="w"> </span><span class="s2">"malachigriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"facebook_profile"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"linkedin_profile"</span><span class="p">:</span><span class="w"> </span><span class="s2">"malachigriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"bio"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Dr. Griffith is an Assistant Professor of Genetics and Assistant Director of the McDonnell Genome Institute at Washington University School of Medicine. Dr Griffith has extensive experience in the fields of genomics, bioinformatics, data mining, and cancer research. His research is focused on improving the understanding of cancer biology and the development of personalized medicine strategies for cancer using genomics and informatics technologies. The Griffith lab develops bioinformatics and statistical methods for the analysis of high throughput sequence data and identification of biomarkers for diagnostic, prognostic and drug response prediction. The Griffith lab uses CIViC to interpret variants identified in cases examined by the WASHU Genomics Tumor Board. He is a co-creator of the CIViC resource."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"featured_expert"</span><span class="p">:</span><span class="w"> </span><span class="kc">true</span><span class="p">,</span><span class="w">
      </span><span class="nt">"accepted_license"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"signup_complete"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="nt">"html"</span><span class="p">:</span><span class="w"> </span><span class="s2">"&lt;p&gt;Note that there appears to be some confusion in the literature about the coordinates and nomenclature for WT1 mutations and the selection of representative transcripts.  This may stem in part from the fact that The WT1 gene encodes numerous protein isoforms. Two translational start sites have been identified, a standard methionine and a non-AUG leucine, which adds 68 amino-terminal residues (&lt;a href=\"http://genesdev.cshlp.org/content/12/20/3217.full\"&gt;http://genesdev.cshlp.org/content/12/20/3217.full&lt;/a&gt;).  This confusion is propagated by the use of ambiguous terms Exon 7 and Exon 9 mutations in the literature, often used without additional information to help reduce ambiguity.  To help clarify these variants, here are some representative primer sequences used to amplify Exon 7 and Exon 9 of WT1 (&lt;a href=\"http://www.ncbi.nlm.nih.gov/pubmed/19221039\"&gt;http://www.ncbi.nlm.nih.gov/pubmed/19221039&lt;/a&gt;).&lt;/p&gt;\n\n&lt;pre&gt;&lt;code&gt;&amp;gt;7F\nCTCCAGTGCTCACTCTCCCTC\n&amp;gt;7R\nCCTTAGCAGTGTGAGAGCCTG\n&amp;gt;9F\nGTGAGGCAGATGCAGACATTG\n&amp;gt;9R\nAGCCACGCACTATTCCTTCTC\n&lt;/code&gt;&lt;/pre&gt;\n"</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">249</span><span class="p">,</span><span class="w">
    </span><span class="nt">"text"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Renneville et al., 2009, Cancer (http://www.ncbi.nlm.nih.gov/pubmed/19536888)\n\n    &gt;7F-2\n    CAGTGCTCACTCTCCCTCAA\n    &gt;7R-2 \n    TAGCAGTGTGAGAGCCTGGA\n    &gt;9F-2\n    TGCAGACATTGCAGGCATGGCAGG\n    &gt;9R-2\n    GCACTATTCCTTCTCTCAACTGAG-3\n\nVirappane et al., 2008, J. Clin. Oncol. (http://www.ncbi.nlm.nih.gov/pubmed/18591546)\n\n    &gt;7F-3\n    GACCTACGTGAATGTTCACATG\n    &gt;7R-3\n    ACCAACACCTGGATCAGACCT\n    &gt;9F-3\n    TGCAGACATTGCAGGCATGGCAGG\n    &gt;9R-3\n    GCACTATTCCTTCTCTCAACTGAG"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"title"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"created_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-09-03T01:16:55.412Z"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"updated_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-09-03T01:16:55.412Z"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"user"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">15</span><span class="p">,</span><span class="w">
      </span><span class="nt">"email"</span><span class="p">:</span><span class="w"> </span><span class="s2">"malachig@gmail.com"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Malachi Griffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"last_seen_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2017-01-05T21:29:19.166Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"username"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MalachiGriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"role"</span><span class="p">:</span><span class="w"> </span><span class="s2">"admin"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatar_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatars"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"x128"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=128"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x64"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=64"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x32"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x14"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=14"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"area_of_expertise"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Research Scientist"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"orcid"</span><span class="p">:</span><span class="w"> </span><span class="s2">"0000-0002-6388-446X"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MalachiGriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"created_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-02-26T22:25:34.692Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://genome.wustl.edu/people/individual/malachi-griffith/"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"twitter_handle"</span><span class="p">:</span><span class="w"> </span><span class="s2">"malachigriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"facebook_profile"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"linkedin_profile"</span><span class="p">:</span><span class="w"> </span><span class="s2">"malachigriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"bio"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Dr. Griffith is an Assistant Professor of Genetics and Assistant Director of the McDonnell Genome Institute at Washington University School of Medicine. Dr Griffith has extensive experience in the fields of genomics, bioinformatics, data mining, and cancer research. His research is focused on improving the understanding of cancer biology and the development of personalized medicine strategies for cancer using genomics and informatics technologies. The Griffith lab develops bioinformatics and statistical methods for the analysis of high throughput sequence data and identification of biomarkers for diagnostic, prognostic and drug response prediction. The Griffith lab uses CIViC to interpret variants identified in cases examined by the WASHU Genomics Tumor Board. He is a co-creator of the CIViC resource."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"featured_expert"</span><span class="p">:</span><span class="w"> </span><span class="kc">true</span><span class="p">,</span><span class="w">
      </span><span class="nt">"accepted_license"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"signup_complete"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="nt">"html"</span><span class="p">:</span><span class="w"> </span><span class="s2">"&lt;p&gt;Renneville et al., 2009, Cancer (&lt;a href=\"http://www.ncbi.nlm.nih.gov/pubmed/19536888\"&gt;http://www.ncbi.nlm.nih.gov/pubmed/19536888&lt;/a&gt;)&lt;/p&gt;\n\n&lt;pre&gt;&lt;code&gt;&amp;gt;7F-2\nCAGTGCTCACTCTCCCTCAA\n&amp;gt;7R-2 \nTAGCAGTGTGAGAGCCTGGA\n&amp;gt;9F-2\nTGCAGACATTGCAGGCATGGCAGG\n&amp;gt;9R-2\nGCACTATTCCTTCTCTCAACTGAG-3\n&lt;/code&gt;&lt;/pre&gt;\n\n&lt;p&gt;Virappane et al., 2008, J. Clin. Oncol. (&lt;a href=\"http://www.ncbi.nlm.nih.gov/pubmed/18591546\"&gt;http://www.ncbi.nlm.nih.gov/pubmed/18591546&lt;/a&gt;)&lt;/p&gt;\n\n&lt;pre&gt;&lt;code&gt;&amp;gt;7F-3\nGACCTACGTGAATGTTCACATG\n&amp;gt;7R-3\nACCAACACCTGGATCAGACCT\n&amp;gt;9F-3\nTGCAGACATTGCAGGCATGGCAGG\n&amp;gt;9R-3\nGCACTATTCCTTCTCTCAACTGAG\n&lt;/code&gt;&lt;/pre&gt;\n"</span><span class="w">
  </span><span class="p">},</span><span class="w">
  </span><span class="p">{</span><span class="w">
    </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">250</span><span class="p">,</span><span class="w">
    </span><span class="nt">"text"</span><span class="p">:</span><span class="w"> </span><span class="s2">"It looks like the three papers being used to obtain evidence items are possibly referring to a WT1 protein sequence that looks like this one:\nhttp://www.uniprot.org/blast/?about=P19544-1.  Renneville et al., 2009, Cancer (http://www.ncbi.nlm.nih.gov/pubmed/19536888) actually refers to P19544 and NM_024426."</span><span class="p">,</span><span class="w">
    </span><span class="nt">"title"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"created_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-09-03T01:33:55.202Z"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"updated_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-09-03T01:33:55.202Z"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"user"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
      </span><span class="nt">"id"</span><span class="p">:</span><span class="w"> </span><span class="mi">15</span><span class="p">,</span><span class="w">
      </span><span class="nt">"email"</span><span class="p">:</span><span class="w"> </span><span class="s2">"malachig@gmail.com"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Malachi Griffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"last_seen_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2017-01-05T21:29:19.166Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"username"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MalachiGriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"role"</span><span class="p">:</span><span class="w"> </span><span class="s2">"admin"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatar_url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"avatars"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span><span class="w">
        </span><span class="nt">"x128"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=128"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x64"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=64"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x32"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=32"</span><span class="p">,</span><span class="w">
        </span><span class="nt">"x14"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://secure.gravatar.com/avatar/a4d9fc3b05d58cf3d3ba51dc30bb61d6.png?d=identicon&amp;r=pg&amp;s=14"</span><span class="w">
      </span><span class="p">},</span><span class="w">
      </span><span class="nt">"area_of_expertise"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Research Scientist"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"orcid"</span><span class="p">:</span><span class="w"> </span><span class="s2">"0000-0002-6388-446X"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"display_name"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MalachiGriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"created_at"</span><span class="p">:</span><span class="w"> </span><span class="s2">"2015-02-26T22:25:34.692Z"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"url"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://genome.wustl.edu/people/individual/malachi-griffith/"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"twitter_handle"</span><span class="p">:</span><span class="w"> </span><span class="s2">"malachigriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"facebook_profile"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"linkedin_profile"</span><span class="p">:</span><span class="w"> </span><span class="s2">"malachigriffith"</span><span class="p">,</span><span class="w">
      </span><span class="nt">"bio"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Dr. Griffith is an Assistant Professor of Genetics and Assistant Director of the McDonnell Genome Institute at Washington University School of Medicine. Dr Griffith has extensive experience in the fields of genomics, bioinformatics, data mining, and cancer research. His research is focused on improving the understanding of cancer biology and the development of personalized medicine strategies for cancer using genomics and informatics technologies. The Griffith lab develops bioinformatics and statistical methods for the analysis of high throughput sequence data and identification of biomarkers for diagnostic, prognostic and drug response prediction. The Griffith lab uses CIViC to interpret variants identified in cases examined by the WASHU Genomics Tumor Board. He is a co-creator of the CIViC resource."</span><span class="p">,</span><span class="w">
      </span><span class="nt">"featured_expert"</span><span class="p">:</span><span class="w"> </span><span class="kc">true</span><span class="p">,</span><span class="w">
      </span><span class="nt">"accepted_license"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
      </span><span class="nt">"signup_complete"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="w">
    </span><span class="p">},</span><span class="w">
    </span><span class="nt">"html"</span><span class="p">:</span><span class="w"> </span><span class="s2">"&lt;p&gt;It looks like the three papers being used to obtain evidence items are possibly referring to a WT1 protein sequence that looks like this one:\n&lt;a href=\"http://www.uniprot.org/blast/?about=P19544-1\"&gt;http://www.uniprot.org/blast/?about=P19544-1&lt;/a&gt;.  Renneville et al., 2009, Cancer (&lt;a href=\"http://www.ncbi.nlm.nih.gov/pubmed/19536888\"&gt;http://www.ncbi.nlm.nih.gov/pubmed/19536888&lt;/a&gt;) actually refers to P19544 and NM_024426.&lt;/p&gt;\n"</span><span class="w">
  </span><span class="p">}</span><span class="w">
</span><span class="p">]</span><span class="w">

</span></code></pre>
<table><thead>
<tr>
<th>Endpoint</th>
<th>Purpose</th>
<th>Example</th>
</tr>
</thead><tbody>
<tr>
<td><code class="prettyprint">/comments</code></td>
<td>View discussion for an entity</td>
<td>/api/genes/1/comments</td>
</tr>
<tr>
<td><code class="prettyprint">/suggested_changes</code></td>
<td>View proposed revisions for an entity</td>
<td>/api/genes/6/suggested_changes</td>
</tr>
<tr>
<td><code class="prettyprint">/revisions</code></td>
<td>View the entire revision history for an entity.</td>
<td>/api/genes/6/revisions</td>
</tr>
</tbody></table>

<h3 id="supported-endpoints-for-civic-entities">Supported endpoints for CIViC entities</h3>

<table><thead>
<tr>
<th>CIViC Entity</th>
<th>Base Path</th>
<th>Supported Endpoints</th>
</tr>
</thead><tbody>
<tr>
<td>Genes</td>
<td><code class="prettyprint">/api/genes/:gene_id</code></td>
<td><code class="prettyprint">/comments</code>. <code class="prettyprint">/suggested_changes</code>, <code class="prettyprint">/revisions</code>, <code class="prettyprint">/variants</code></td>
</tr>
<tr>
<td>Variants</td>
<td><code class="prettyprint">/api/variants/:variant_id</code></td>
<td><code class="prettyprint">/comments</code>. <code class="prettyprint">/suggested_changes</code>, <code class="prettyprint">/revisions</code>, <code class="prettyprint">/evidence_items</code></td>
</tr>
<tr>
<td>Evidence Items</td>
<td><code class="prettyprint">/api/evidence_items/:eid</code></td>
<td><code class="prettyprint">/comments</code>. <code class="prettyprint">/suggested_changes</code>, <code class="prettyprint">/revisions</code></td>
</tr>
<tr>
<td>Variant Groups</td>
<td><code class="prettyprint">/api/variant_groups/:variant_group_id</code></td>
<td><code class="prettyprint">/comments</code>. <code class="prettyprint">/suggested_changes</code>, <code class="prettyprint">/revisions</code></td>
</tr>
</tbody></table>

      </div>
      <div class="dark-box">
          <div class="lang-selector">
                <a href="#" data-language-name="shell">Request</a>
                <a href="#" data-language-name="json">Response</a>
          </div>
      </div>
    </div>
  </body>
</html>
